Started by user [8mha:////4C8zTO/v0UlG+vQ5Fjm5jTf9kR93yL5yxTMP2X04T9v8AAAAlh+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAyWEgYu/dLi1CL9ovz8EgDKVWzSvwAAAA==[0mroot
Running in Durability level: MAX_SURVIVABILITY
[8mha:////4DOlrZhma2TKYqx8mtRBwGXVE8Xt4z3OJyo/zi+ICDdzAAAAoh+LCAAAAAAAAP9tjTEOwjAQBM8BClpKHuFItIiK1krDC0x8GCfWnbEdkooX8TX+gCESFVvtrLSa5wtWKcKBo5UdUu8otU4GP9jS5Mixv3geZcdn2TIl9igbHBs2eJyx4YwwR1SwULBGaj0nRzbDRnX6rmuvydanHMu2V1A5c4MHCFXMWcf8hSnC9jqYxPTz/BXAFEIGsfuclm8zQVqFvQAAAA==[0m[Pipeline] Start of Pipeline
[8mha:////4B4K7nOhvaaLmbeE3w0NEhE1GAALwLDjakw9plXgkY6XAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycohUghExsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jduZBmjwAAAAA==[0m[Pipeline] node
Running on [8mha:////4C8PgMHPegRpCg8+HpIiIHK02uxvuQktJylkqRHkJagNAAAAox+LCAAAAAAAAP9b85aBtbiIQTGjNKU4P08vOT+vOD8nVc83PyU1x6OyILUoJzMv2y+/JJUBAhiZGBgqihhk0NSjKDWzXb3RdlLBUSYGJk8GtpzUvPSSDB8G5tKinBIGIZ+sxLJE/ZzEvHT94JKizLx0a6BxUmjGOUNodHsLgAzeEgYJ/eT83ILSktQiICOvpCg/Jye1SNfSUh8ABjlyb80AAAA=[0mcontroller-99 in /root/.jenkins/workspace/icx-d-test-nic-sriov-5times
[8mha:////4NUxBMzN6CtFAbQ4pdo7pH8ZuwVHigLf0SdP3FcjQTwyAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gA0xsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jfoP95RwAAAAA==[0m[Pipeline] {
[8mha:////4OCB1+Up79PmV3LnMeomKr9r+JFCKkFjAfTpr2UorbLOAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gQkxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jc09154wAAAAA==[0m[Pipeline] stage
[8mha:////4N6KQN35K8CT/Pcr9a+R4oOEu4uEh3phvpB1t67zd2hmAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0ggUxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jek7ggRwAAAAA==[0m[Pipeline] { (rm project dir)
[8mha:////4EWjgIQS1R18e39T6LdOu3ZpSMSXMburmKwxJHd8e163AAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0gwExsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jcChmMxwAAAAA==[0m[Pipeline] script
[8mha:////4NOJ/YVWDlnnBpjDdLKlLZkF5LrR387vblsx7fWhZkPlAAAApR+LCAAAAAAAAP9tjTEOwjAUQ3+KOrAycoh0BAkxsUZZOEFIQkgb/d8mKe3EibgadyBQiQlLlmxL1nu+oE4RjhQdby12HpP2vA+jK4lPFLtroIm3dOGaMFGwXNpJkrGnpUrKFhaxClYC1hZ1oOTRZdiIVt1VExS65pxj2Q4CKm8GeAAThZxVzN8yR9jeRpMIf5y/AJj7DGxXvP/86jfpX/cvwAAAAA==[0m[Pipeline] {
[8mha:////4MAm0moKTc8Eo2PcB0LgkHNHI7Xtf5Poej2Mf/l8xh/lAAAAoh+LCAAAAAAAAP9tjTEOAiEURD9rLGwtPQTbaoyVLaHxBMgiwpL/WWDdrTyRV/MOEjexcpJJ5k3zXm9Y5wQnSpZ7g73DrB2PYbR18YlSfws0cU9XrgkzBcOlmSR15rygpGJgCWtgJWBjUAfKDm2BrfDqodqg0LaXkup3FNC4boAnMFHNRaXyhTnB7j52mfDn+SuAORZg+9pD/AAeoCAqvAAAAA==[0m[Pipeline] sh
+ ls /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester
1.0.0.0001.version
conf
installation
libs
log_analyzer
README.md
testconf
testLibs
Tests
[8mha:////4F3BY6hez6Yn9lSF4dYULOlz4RwK0Fdsp1OYWtj0vxDtAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMKBDglR0VppeIFJjHFi3QX7QlLxIr7GH4iIRMVWO9PM641lijhydKqx1HpKlVdd6N301MCxvQYeVMMXVTElDlaVdii5tqcZSxaLeVmOhcbKUhU4eXKCtW7MwxTBkCvOEid30Mh9fccTmZ7KYqJ8YYzY3Po6Mf06fwMYO0G2F+S7bfcBITL9lL0AAAA=[0m[Pipeline] echo
dir is exists and rm dir
[8mha:////4KL1cTFlC/t8/0LRf71hJwxXIUFLIMviWtcIHXR2W8epAAAAox+LCAAAAAAAAP9tjbEOgjAURS8YB1dHP6KEzcQ4uTYsfkGFWgvNe9g+hMkv8tf8B4kkTt7pnrOc1xvrFHHk6FRrqfOUaq/6MLj5qZFjdw08qpYvqmZKHKyq7FhxY08LViwWy7IcK42NpTpw8uQEW92ahymCIVecJc7uoJH75o4nMj2XxUT5whSxuw1NYvp1/gYw9YJsL8jLsv8AYAPmjb0AAAA=[0m[Pipeline] sh
+ rm -rf /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester
[8mha:////4GzJynz9IX3CLl34e/1JpYOeLz7FDsqMtIihCtFGqNpZAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRVEiIitZKwwtMYowT6y7YF5KKF/E1/kBEJCq22plmXm8sU8SRo1ONpdZTqrzqQu+mpwaO7TXwoBq+qIopcbCqtEPJtT3NWLJYzMtyLDRWlqrAyZMTrHVjHqYIhlxxlji5g0bu6zueyPRUFhPlC2PE5tbXienX+RvA2HWCfLsTZPsPzjQzob0AAAA=[0m[Pipeline] }
[8mha:////4MvApYhNLg1xEFG2iXyHCog7Z9h4MbdAsNSDoK4wJnDgAAAApB+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIRoqBAVLRWGl5gEmOcWHfBvpBUvIiv8QciIlGx1c4083pjmSKOHJ1qLLWeUuVVF3o3PTVwbK+BB9XwRVVMiYNVpR1Kru1pxpLFYl6WY6GxslQFTp6cYK0b8zBFMOSKs8TJHTRyX9/xRKanspgoXxgjNre+Tky/zt8Axq4T5NudINt/AO8A7Ay9AAAA[0m[Pipeline] // script
[8mha:////4P1g47DHQeYIMdAQPrqVFJW5XTEYSJkMqP0k+p1upCWxAAAApB+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIREqJBVLRWGl5gEmOcWHfBvpBUvIiv8QciIlGx1c4083pjmSKOHJ1qLLWeUuVVF3o3PTVwbK+BB9XwRVVMiYNVpR1Kru1pxpLFYl6WY6GxslQFTp6cYK0b8zBFMOSKs8TJHTRyX9/xRKanspgoXxgjNre+Tky/zt8Axq4T5NudINt/AGnsy8m9AAAA[0m[Pipeline] }
[8mha:////4Lc6WrLYEiiAC7rTWY8q4RQPGu1FD+hrgClvEqpzcg2sAAAAox+LCAAAAAAAAP9tjTESgjAQRT84FraWHiKMBZVjZZuh8QQRYgxkdjFZhMoTeTXvICMzVv7qv9e81xvrFHHk6FRrqfOUaq/6MLj5qZFjdw08qpYvqmZKHKyq7FhxY08LViwWy7IcK42NpTpw8uQEW92ahymCIVecJc7uoJH75o4nMj2XxUT5whSxuw1NYvp1/gYw9b0g35eCrPwAY5Sibb0AAAA=[0m[Pipeline] // stage
[8mha:////4LPzYIRsJlP9vBl87WR2FbvV5590n9jZkqHCRBijPEs5AAAApx+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjJESDqGgtN7zAxMY4se6CfSGpeBFf4w8EIlGx1c5Kq3m+oMwJDpS8bBy2AXMdZBd7PzU5UGovkQbZ0FnWhJmik9oNmqw7zqiJHcwRBSwULB3WkXJAz7BSjbmbKhr01YnTtO0VFMHe4AFCTWY2ib8wJlhfe5sJf56/Ahg7BrFlKDa7z7F8A6sHc4fBAAAA[0m[Pipeline] stage
[8mha:////4P0yjWfQiEwQxZO7i9obwDXrYwQclNR3i//1CTGbFwBFAAAApx+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohULDAgpq5RFk4QmhDSRv+Xn5R24kRcjTvQUokJD5ZtyXqvNxSR4UTsRGOx9RhrL7rQuymJgbi9BhpEQxdRE0YKVig7KDK2WqqiZGFRlsNKwtpiHSh6dAk2stEPXQaNrjwnnrajhNybOzwhkxM5aU7fMjJsb72JhD/OXwCMXYJ8t5/tMD+LD1+IXOLCAAAA[0m[Pipeline] { (git clone)
[8mha:////4I3tKNYAz7At397ZkNOs6SFwEn2bzn3ZBhsoegW1dhRwAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiKpAQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGPIUG53n9qHNxXfVKy+AAAA[0m[Pipeline] sh
+ git clone -b develop https://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@github.com/Jiz1028/applications.validation.network-platform.npxtester.git npxtester
Cloning into 'npxtester'...
[8mha:////4CRWXV91kUhqh2bTVXI+2n5U32cQ89nJ7yub5blKQ29bAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiKoQQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGMIGcrt/lO7N9Jjga++AAAA[0m[Pipeline] }
[8mha:////4I4ZVrOFMvh2sVfxnfT9s6+paGHoH9OzkxEwdR4h+nRDAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOICFDQoFa3lhheYxBgn1l2wHZyKF/E1/oBFJCq2WO1sM683LIOHmr3BTlNvKTQWBzeavDCx76+OE3Z8wYYpsNModZLc6tOMkqOGOUUJCwErTY3jYMlEWItOPVTlFJnqHH3+jgJK297hCYXI5qh8/MLkYXMb28D08/wVwDQMEcr9Ntfu8AERGGHOvgAAAA==[0m[Pipeline] // stage
[8mha:////4J4BpfnRrLE6odVrNZGgZ11wIRy0HYHBalvcHAcD12j/AAAAph+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjEB2iorXc8AITG+PEugv2haTiRXyNPxCIRMVWOyut5vmCMic4UPKycdgGzHWQXez91ORAqb1EGmRDZ1kTZopOajdosu44oyZ2MEcUsFCwdFhHygE9w0o15m6qaNBXJ07TtldQBHuDBwg1mdkk/sKYYH3tbSb8ef4KYOwYxI6h2G4+x/INji2eq8EAAAA=[0m[Pipeline] stage
[8mha:////4AgPhGmio4pMNzq5gItmuDxswb3P5dwyIofUIBdMXBTmAAAAph+LCAAAAAAAAP9tjTEOwjAUQ3+DOrAycohUdEVMrFEXThCaEJJG/7dJSjpxIq7GHWipxIQHy7ZkvdcbyhjgRMFwp7GzGFvLez+aOfFMobt5ytzRlbeEkbzmjc4NKX1ea0NJw6qCwUbAVmPrKVo0CXbCyYesvERTXVKYt6MAZtUATyjETE4ypG+ZAuzvo4qEP85fAEx9AlYfFquXZ/kBAsTc3cIAAAA=[0m[Pipeline] { (pre-exec-set-args)
[8mha:////4G9dxluK5ut2lxdfo2JVi6OIov1ZH/881VUFzuPzanonAAAAox+LCAAAAAAAAP9tjTESwiAURH/iWNhaeggyE0vHypah8QSYIEKY/wkQSeWJvJp3EM2MlVvs7NvmPV+wjgGOFDSzCgeDsTPMu0mXxTKF4eooM0sX1hFGcooJlQX16rSgoKRgSVXDisNGYecoGtQJttzKu2ycRN2cUyjfgUNt+hEeUPFiTjKkL8wBdrepj4Q/z18BzD5B3baf2vs3w5BRkb4AAAA=[0m[Pipeline] sh
+ python /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/conf/change_args.py '{"drive_info":{"drive_version":"1.7.19"},"dpdk_dir":"/opt/APP/utility/DPDK/dpdk-21.11/","vm_image_conf":{"image1":"uvm1.img","image2":"uvm2.img","image3":"rhel84rt.img","image4":"rhel84rt2.img","image_dir":"/home/image"},"sut_conf":{"host":"10.239.115.71","pf1":"enp137s0f0","os":"UBUNTU"},"pkg_generator":{"host":"10.239.115.16","pf1":"ens801f0"}}'
[8mha:////4FhUQONKPXFSeOjFO92QKbtxv93cUzSuhB7hx3xoWX6KAAAApB+LCAAAAAAAAP9tjbEOwiAURV9rHFwd/QiaNG7GyZWw+AXYIkLJexSodPKL/DX/QbSJk3e4uecu5/mCdQxwpKCZVTgYjJ1h3k26LJYpDFdHmVm6sI4wklNMqCyoV6cFBSUFS6oaVhw2CjtH0aBOsOVW3mXjJOrmnEL5Dhxq04/wgIoXc5IhfWEOsLtNfST8ef4KYPY+Qd3uP9W+Ad3NsHm+AAAA[0m[Pipeline] }
[8mha:////4EpM1Tihe33kOTBucrClpdwMCq1FF6wGwTU/AEOZmYaNAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIikKgQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGMIGcrd/lPbN8JP5Su+AAAA[0m[Pipeline] // stage
[8mha:////4JJk4NQMkVX21N+ktknOqKLQw+FD2VU/fvD8dhBPuambAAAApx+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjIUSDqGgtN7zAxMY4se6CfSGpeBFf4w8EIlGx1c5Kq3m+oMwJDpS8bBy2AXMdZBd7PzU5UGovkQbZ0FnWhJmik9oNmqw7zqiJHcwRBSwULB3WkXJAz7BSjbmbKhr01YnTtO0VFMHe4AFCTWY2ib8wJlhfe5sJf56/Ahg7BrFlKDa7z7F8AzYdm7bBAAAA[0m[Pipeline] stage
[8mha:////4GToWcz1lEtttcc1F67+obVKf97ChyGWoNBgdNb8ln6iAAAAqB+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohUYoABMXWNsnCC0ISQNvq//KS0EyfiatyBlkpMeLBsS9Z7vaGIDCdiJxqLrcdYe9GF3k1JDMTtNdAgGrqImjBSsELZQZGx1VIVJQuLshxWEtYW60DRo0uwkY1+6DJodOU58bQdJeTe3OEJmZzISXP6lpFhe+tNJPxx/gJg7BLku/1sh/lZfAAyQCqkwgAAAA==[0m[Pipeline] { (run nic case)
[8mha:////4M/BBJOrvLliK1a64sS7A0hf4H4seiclg8D1IX9OqKAVAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIi0YCEqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDhnK7+9Q+vAGvZp0gvgAAAA==[0m[Pipeline] sh
+ pytest /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics -s -m nicSriov5times --capture=tee-sys --html=/root/testconfig/testlogs/HtmlReport/nic_sriov5times_case_report.html --self-contained-html
============================= test session starts ==============================
platform linux -- Python 3.6.8, pytest-6.2.4, py-1.11.0, pluggy-0.13.1
rootdir: /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics, configfile: pytest.ini
plugins: metadata-1.11.0, html-3.1.1
collected 17 items / 16 deselected / 1 selected

npxtester/Tests/testNics/test_network_sriov_stress.py 2022-03-21 11:10:35 - NicTestFixture.py - [line:30] - root - INFO - ============================ Starting to run test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[uvm2.img-uvm1.img] case ============================
2022-03-21 11:10:35 - test_network_sriov_stress.py - [line:28] - root - INFO - SSH SUT and setup configurations in the setup function.
2022-03-21 11:10:35 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.2p1)
2022-03-21 11:10:35 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-03-21 11:10:35 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.115.71
2022-03-21 11:10:35 - sftp.py - [line:158] - paramiko.transport.sftp - INFO - [chan 1] Opened sftp connection (server version 3)
2022-03-21 11:10:35 - transport.py - [line:1819] - paramiko.transport - INFO - Connected (version 2.0, client OpenSSH_8.0)
2022-03-21 11:10:35 - transport.py - [line:1819] - paramiko.transport - INFO - Authentication (password) successful!
2022-03-21 11:10:35 - sshInstance.py - [line:28] - root - INFO - SSH on SUT: 10.239.115.16
2022-03-21 11:10:38 - sshInstance.py - [line:46] - root - INFO - modinfo ice | grep ^version:
2022-03-21 11:10:38 - sshInstance.py - [line:50] - root - INFO - version:        1.7.19

2022-03-21 11:10:46 - sshInstance.py - [line:46] - root - INFO - rmmod ice
2022-03-21 11:10:49 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:10:57 - sshInstance.py - [line:46] - root - INFO - modprobe ice
2022-03-21 11:11:02 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:11:10 - sshInstance.py - [line:46] - root - INFO - ifconfig enp137s0f0 up
2022-03-21 11:11:10 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:11:18 - sshInstance.py - [line:46] - root - INFO - ethtool -i enp137s0f0
2022-03-21 11:11:18 - sshInstance.py - [line:50] - root - INFO - driver: ice
version: 1.7.19
firmware-version: 2.19 0x8000bafe 1.3083.0
expansion-rom-version: 
bus-info: 0000:89:00.0
supports-statistics: yes
supports-test: yes
supports-eeprom-access: yes
supports-register-dump: yes
supports-priv-flags: yes

2022-03-21 11:11:26 - sshInstance.py - [line:46] - root - INFO - ifconfig enp137s0f0
2022-03-21 11:11:26 - sshInstance.py - [line:50] - root - INFO - enp137s0f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::200:ff:fe00:3300  prefixlen 64  scopeid 0x20<link>
        ether 00:00:00:00:33:00  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 12  bytes 2036 (2.0 KB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-03-21 11:11:34 - sshInstance.py - [line:46] - root - INFO - ifconfig ens801f0
2022-03-21 11:11:34 - sshInstance.py - [line:50] - root - INFO - ens801f0: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet6 fe80::3efd:feff:fed2:73e0  prefixlen 64  scopeid 0x20<link>
        ether 3c:fd:fe:d2:73:e0  txqueuelen 1000  (Ethernet)
        RX packets 9759985555  bytes 14765465564501 (13.4 TiB)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 413079860  bytes 27353018271 (25.4 GiB)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0


2022-03-21 11:11:39 - test_network_sriov_stress.py - [line:388] - root - INFO - 

********************************************************Stress Disable and Enable for 1 time when VF is Assigned VM guest.*************************************************************


2022-03-21 11:11:42 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK/dpdk-21.11/usertools/dpdk-devbind.py -s | grep enp137s0f0 | awk '{print $1;}'
2022-03-21 11:11:42 - sshInstance.py - [line:50] - root - INFO - 0000:89:00.0

2022-03-21 11:11:47 - test_network_sriov_stress.py - [line:111] - root - INFO - pf_id was found: 0000:89:00.0

2022-03-21 11:11:47 - test_network_sriov_stress.py - [line:396] - root - INFO - format_pf_id is 0000\:89\:00.0

2022-03-21 11:11:47 - test_network_sriov_stress.py - [line:399] - root - INFO - pf_id_key: 89
2022-03-21 11:11:53 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:11:53 - sshInstance.py - [line:96] - root - INFO - Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
Last login: Mon Mar 21 11:10:22 2022 from 10.249.160.143
root@npx-sut3-ice-d-auto:~# ifconfig enp137s0f0 192.168.10.10
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:11:56 - test_network_sriov_stress.py - [line:302] - root - INFO - Successfully assigned ip to ethernet: enp137s0f0
2022-03-21 11:12:02 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:12:02 - sshInstance.py - [line:96] - root - INFO - Activate the web console with: systemctl enable --now cockpit.socket
This system is not registered to Red Hat Insights. See https://cloud.redhat.com/
To register this system, run: insights-client --register
Last login: Mon Mar 21 10:51:48 2022 from 10.239.182.99
[root@redhat8 ~]# ifconfig ens801f0 192.168.10.11
[root@redhat8 ~]# 
2022-03-21 11:12:05 - test_network_sriov_stress.py - [line:302] - root - INFO - Successfully assigned ip to ethernet: ens801f0
2022-03-21 11:12:08 - sshInstance.py - [line:46] - root - INFO - echo 4 > /sys/class/net/enp137s0f0/device/sriov_numvfs
2022-03-21 11:12:08 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:12:13 - test_network_sriov_stress.py - [line:125] - root - INFO - Successfully Virtualized PF to VFs.
2022-03-21 11:12:16 - sshInstance.py - [line:46] - root - INFO -  lspci | grep "Virtual" | awk '{print $1;}'list
2022-03-21 11:12:16 - sshInstance.py - [line:50] - root - INFO - 89:01.0
89:01.1
89:01.2
89:01.3

2022-03-21 11:12:21 - test_network_sriov_stress.py - [line:139] - root - INFO - Succssfully Gotten vf_id list:

2022-03-21 11:12:24 - sshInstance.py - [line:46] - root - INFO - modprobe vfio-pci

2022-03-21 11:12:24 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:12:32 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK/dpdk-21.11/usertools/dpdk-devbind.py -b vfio-pci 89:01.0
2022-03-21 11:12:33 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:12:38 - test_network_sriov_stress.py - [line:172] - root - INFO - Successfuly bound VF to vfio-pci
2022-03-21 11:12:41 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK/dpdk-21.11/usertools/dpdk-devbind.py -b vfio-pci 89:01.1
2022-03-21 11:12:42 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:12:47 - test_network_sriov_stress.py - [line:172] - root - INFO - Successfuly bound VF to vfio-pci
2022-03-21 11:12:47 - test_network_sriov_stress.py - [line:421] - root - INFO - lspci -Dnn | grep 89:01.1
2022-03-21 11:12:50 - sshInstance.py - [line:46] - root - INFO - lspci -Dnn | grep 89:01.1
2022-03-21 11:12:50 - sshInstance.py - [line:50] - root - INFO - 0000:89:01.1 Ethernet controller [0200]: Intel Corporation Ethernet Adaptive Virtual Function [8086:1889]

2022-03-21 11:12:55 - test_network_sriov_stress.py - [line:433] - root - INFO - Successfully get vf device id: 8086:1889
2022-03-21 11:12:58 - sshInstance.py - [line:46] - root - INFO - virsh net-info default | grep "Active" | awk '{print $2;}'
2022-03-21 11:12:58 - sshInstance.py - [line:50] - root - INFO - yes

2022-03-21 11:13:03 - test_network_sriov_stress.py - [line:198] - root - INFO - Default network is active? yes

2022-03-21 11:13:07 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vvcpus=5 --network network=default --hostdev pci_0000_89_01_0 --boot hd --disk patth=/home/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=55900 --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:89:01.0 is in use by driver QEMU, domain VMtest1
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest1
otherwise, please restart your installation.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:13:07 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:13:10 - test_network_sriov_stress.py - [line:230] - root - ERROR - Failed to create a VM: virt-install --name VMtest1 --ram 4096 --cpu host --vvcpus=5 --network network=default --hostdev pci_0000_89_01_0 --boot hd --disk patth=/home/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=55900 --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:89:01.0 is in use by driver QEMU, domain VMtest1
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest1
otherwise, please restart your installation.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:13:10 - test_network_sriov_stress.py - [line:231] - root - ERROR - Failed to create a VM: 
2022-03-21 11:13:14 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vvcpus=5 --network network=default --hostdev pci_0000_89_01_0 --boot hd --disk patth=/home/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=55900 --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-03-21 11:13:16 - sshInstance.py - [line:71] - root - INFO - Domain creation completed.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:13:16 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:13:19 - test_network_sriov_stress.py - [line:233] - root - INFO - Successfully created a VM: VMtest1
2022-03-21 11:13:23 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vvcpus=5 --network network=default --hostdev pci_0000_89_01_1 --boot hd --disk patth=/home/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=55901 --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:89:01.1 is in use by driver QEMU, domain VMtest2
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest2
otherwise, please restart your installation.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:13:23 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:13:26 - test_network_sriov_stress.py - [line:230] - root - ERROR - Failed to create a VM: virt-install --name VMtest2 --ram 4096 --cpu host --vvcpus=5 --network network=default --hostdev pci_0000_89_01_1 --boot hd --disk patth=/home/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=55901 --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:89:01.1 is in use by driver QEMU, domain VMtest2
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest2
otherwise, please restart your installation.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:13:26 - test_network_sriov_stress.py - [line:231] - root - ERROR - Failed to create a VM: 
2022-03-21 11:13:30 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vvcpus=5 --network network=default --hostdev pci_0000_89_01_1 --boot hd --disk patth=/home/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=55901 --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-03-21 11:13:32 - sshInstance.py - [line:71] - root - INFO - Domain creation completed.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:13:32 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:13:35 - test_network_sriov_stress.py - [line:233] - root - INFO - Successfully created a VM: VMtest2
2022-03-21 11:13:38 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-03-21 11:13:38 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:13:43 - test_network_sriov_stress.py - [line:250] - root - ERROR - not found IP of VMtest1
2022-03-21 11:14:46 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-03-21 11:14:46 - sshInstance.py - [line:50] - root - INFO - 192.168.122.128/24

2022-03-21 11:14:51 - test_network_sriov_stress.py - [line:245] - root - INFO - Successfully got VMtest1 IP: 192.168.122.128/24

2022-03-21 11:14:54 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest2 | grep '192.168' | awk '{print $4;}'
2022-03-21 11:14:54 - sshInstance.py - [line:50] - root - INFO - 192.168.122.196/24

2022-03-21 11:14:59 - test_network_sriov_stress.py - [line:245] - root - INFO - Successfully got VMtest2 IP: 192.168.122.196/24

2022-03-21 11:14:59 - test_network_sriov_stress.py - [line:450] - root - INFO - ip of VM1: 192.168.122.128
2022-03-21 11:14:59 - test_network_sriov_stress.py - [line:451] - root - INFO - ip of VM1: 192.168.122.196
2022-03-21 11:15:03 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.128
PING 192.168.122.128 (192.168.122.128) 56(84) bytes of data.
64 bytes from 192.168.122.128: icmp_seq=1 ttl=64 time=0.096 ms
2022-03-21 11:15:04 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=2 ttl=64 time=0.088 ms
2022-03-21 11:15:05 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=3 ttl=64 time=0.086 ms
2022-03-21 11:15:06 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=4 ttl=64 time=0.080 ms
2022-03-21 11:15:07 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=5 ttl=64 time=0.080 ms

--- 192.168.122.128 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4092ms
rtt min/avg/max/mdev = 0.080/0.086/0.096/0.006 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:15:07 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:15:16 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:15:16 - sshInstance.py - [line:96] - root - INFO - ssh root@192.168.122.128
root@192.168.122.128's password: 
2022-03-21 11:15:25 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:15:25 - sshInstance.py - [line:96] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Sun 20 Mar 2022 11:17:12 PM EDT
  System load:  0.07               Processes:             137
  Usage of /:   37.1% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.128
  Swap usage:   0%

363 updates can be installed immediately.
221 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Sun Mar 20 22:34:48 2022 from 192.168.122.1
root@localhost:~# 
2022-03-21 11:15:34 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:15:34 - sshInstance.py - [line:96] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-03-21 11:15:40 - test_network_sriov_stress.py - [line:341] - root - INFO - successfully got vf pci id of vm1: 0000:00:05.0
2022-03-21 11:15:46 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:15:46 - sshInstance.py - [line:96] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-03-21 11:15:51 - test_network_sriov_stress.py - [line:355] - root - INFO - Successfully got vf device name of vm1: ens5
2022-03-21 11:15:57 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:15:57 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 192.168.10.12
root@localhost:~# 
2022-03-21 11:16:00 - test_network_sriov_stress.py - [line:302] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-03-21 11:16:06 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:16:06 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 down
root@localhost:~# 
2022-03-21 11:16:15 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:16:15 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 hw ether 1234567890ac
root@localhost:~# 
2022-03-21 11:16:24 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:16:24 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-03-21 11:16:33 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:16:33 - sshInstance.py - [line:96] - root - INFO - nmcli device connect ens5
Command 'nmcli' not found, but can be installed with:
apt install network-manager
root@localhost:~# 
2022-03-21 11:16:42 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:16:42 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.12  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::1034:56ff:fe78:90ac  prefixlen 64  scopeid 0x20<link>
        ether 12:34:56:78:90:ac  txqueuelen 1000  (Ethernet)
        RX packets 1  bytes 345 (345.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 9  bytes 766 (766.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-03-21 11:16:51 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:16:51 - sshInstance.py - [line:96] - root - INFO - exit
logout
Connection to 192.168.122.128 closed.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:16:58 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.196
PING 192.168.122.196 (192.168.122.196) 56(84) bytes of data.
64 bytes from 192.168.122.196: icmp_seq=1 ttl=64 time=0.082 ms
2022-03-21 11:16:59 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.196: icmp_seq=2 ttl=64 time=0.071 ms
2022-03-21 11:17:00 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.196: icmp_seq=3 ttl=64 time=0.065 ms
2022-03-21 11:17:01 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.196: icmp_seq=4 ttl=64 time=0.060 ms
2022-03-21 11:17:02 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.196: icmp_seq=5 ttl=64 time=0.070 ms

--- 192.168.122.196 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4083ms
rtt min/avg/max/mdev = 0.060/0.069/0.082/0.007 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:17:02 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:17:11 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:17:11 - sshInstance.py - [line:96] - root - INFO - ssh root@192.168.122.196
root@192.168.122.196's password: 
2022-03-21 11:17:20 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:17:20 - sshInstance.py - [line:96] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Sun 20 Mar 2022 11:19:07 PM EDT
  System load:  0.0                Processes:             136
  Usage of /:   37.2% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.196
  Swap usage:   0%

363 updates can be installed immediately.
221 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Sun Mar 20 22:32:31 2022 from 192.168.122.1
root@localhost:~# 
2022-03-21 11:17:29 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:17:29 - sshInstance.py - [line:96] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-03-21 11:17:35 - test_network_sriov_stress.py - [line:341] - root - INFO - successfully got vf pci id of vm1: 0000:00:05.0
2022-03-21 11:17:41 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:17:41 - sshInstance.py - [line:96] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-03-21 11:17:46 - test_network_sriov_stress.py - [line:355] - root - INFO - Successfully got vf device name of vm1: ens5
2022-03-21 11:17:52 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:17:52 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 192.168.10.13
root@localhost:~# 
2022-03-21 11:17:55 - test_network_sriov_stress.py - [line:302] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-03-21 11:18:01 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:18:01 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 down
root@localhost:~# 
2022-03-21 11:18:10 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:18:10 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 hw ether 1234567890ab
root@localhost:~# 
2022-03-21 11:18:19 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:18:19 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-03-21 11:18:28 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:18:28 - sshInstance.py - [line:96] - root - INFO - nmcli device connect ens5
Command 'nmcli' not found, but can be installed with:
apt install network-manager
root@localhost:~# 
2022-03-21 11:18:37 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:18:37 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.13  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::1034:56ff:fe78:90ab  prefixlen 64  scopeid 0x20<link>
        ether 12:34:56:78:90:ab  txqueuelen 1000  (Ethernet)
        RX packets 1  bytes 345 (345.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 9  bytes 766 (766.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-03-21 11:18:46 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:18:46 - sshInstance.py - [line:96] - root - INFO - exit
logout
Connection to 192.168.122.196 closed.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:18:49 - test_network_sriov_stress.py - [line:469] - root - INFO - Ping two vms IP from SUT:
2022-03-21 11:18:53 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.12
PING 192.168.10.12 (192.168.10.12) 56(84) bytes of data.
64 bytes from 192.168.10.12: icmp_seq=1 ttl=64 time=0.316 ms
2022-03-21 11:18:54 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=2 ttl=64 time=0.046 ms
2022-03-21 11:18:55 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=3 ttl=64 time=0.044 ms
2022-03-21 11:18:56 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=4 ttl=64 time=0.037 ms
2022-03-21 11:18:57 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=5 ttl=64 time=0.037 ms

--- 192.168.10.12 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4104ms
rtt min/avg/max/mdev = 0.037/0.096/0.316/0.110 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:18:57 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:19:04 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=0.352 ms
2022-03-21 11:19:05 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.054 ms
2022-03-21 11:19:06 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.033 ms
2022-03-21 11:19:07 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.026 ms
2022-03-21 11:19:08 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.032 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4097ms
rtt min/avg/max/mdev = 0.026/0.099/0.352/0.126 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:19:08 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:19:11 - test_network_sriov_stress.py - [line:477] - root - INFO - ping VMtest2 IP from VMtest1
2022-03-21 11:19:15 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.128
PING 192.168.122.128 (192.168.122.128) 56(84) bytes of data.
64 bytes from 192.168.122.128: icmp_seq=1 ttl=64 time=0.091 ms
2022-03-21 11:19:16 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=2 ttl=64 time=0.084 ms
2022-03-21 11:19:17 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=3 ttl=64 time=0.075 ms
2022-03-21 11:19:18 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=4 ttl=64 time=0.081 ms
2022-03-21 11:19:19 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=5 ttl=64 time=0.084 ms

--- 192.168.122.128 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4089ms
rtt min/avg/max/mdev = 0.075/0.083/0.091/0.005 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:19:19 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:19:28 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:19:28 - sshInstance.py - [line:96] - root - INFO - ssh root@192.168.122.128
root@192.168.122.128's password: 
2022-03-21 11:19:37 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:19:37 - sshInstance.py - [line:96] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Sun 20 Mar 2022 11:21:25 PM EDT
  System load:  0.0                Processes:             130
  Usage of /:   37.1% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.128
  Swap usage:   0%                 IPv4 address for ens5: 192.168.10.12

363 updates can be installed immediately.
221 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Sun Mar 20 23:17:13 2022 from 192.168.122.1
root@localhost:~# 
2022-03-21 11:19:44 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=0.118 ms
2022-03-21 11:19:45 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.037 ms
2022-03-21 11:19:46 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.032 ms
2022-03-21 11:19:47 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.031 ms
2022-03-21 11:19:48 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.031 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4094ms
rtt min/avg/max/mdev = 0.031/0.049/0.118/0.034 ms
root@localhost:~# 
2022-03-21 11:19:48 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:19:57 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:19:57 - sshInstance.py - [line:96] - root - INFO - exit
logout
Connection to 192.168.122.128 closed.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:20:00 - test_network_sriov_stress.py - [line:485] - root - INFO - ping VMtest2 IP from VMtest1
2022-03-21 11:20:04 - sshInstance.py - [line:71] - root - INFO - ping -c 5 -I 192.168.10.10 192.168.10.11
PING 192.168.10.11 (192.168.10.11) from 192.168.10.10 : 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.100 ms
2022-03-21 11:20:06 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.038 ms
2022-03-21 11:20:07 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.038 ms
2022-03-21 11:20:08 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.032 ms
2022-03-21 11:20:09 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.027 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4081ms
rtt min/avg/max/mdev = 0.027/0.047/0.100/0.026 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:20:09 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:20:15 - sshInstance.py - [line:46] - root - INFO - echo 0 > /sys/class/net/enp137s0f0/device/sriov_numvfs
2022-03-21 11:20:17 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:20:22 - test_network_sriov_stress.py - [line:264] - root - INFO - Successfully removed VFs from PF.
2022-03-21 11:20:25 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest1
2022-03-21 11:20:26 - sshInstance.py - [line:50] - root - INFO - Domain VMtest1 destroyed


2022-03-21 11:20:31 - test_network_sriov_stress.py - [line:277] - root - INFO - Successfully shut down the VM: VMtest1
2022-03-21 11:20:34 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest1
2022-03-21 11:20:34 - sshInstance.py - [line:50] - root - INFO - Domain VMtest1 has been undefined


2022-03-21 11:20:39 - test_network_sriov_stress.py - [line:287] - root - INFO - Successfully removed the VM: VMtest1
2022-03-21 11:20:42 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest2
2022-03-21 11:20:42 - sshInstance.py - [line:50] - root - INFO - Domain VMtest2 destroyed


2022-03-21 11:20:47 - test_network_sriov_stress.py - [line:277] - root - INFO - Successfully shut down the VM: VMtest2
2022-03-21 11:20:50 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest2
2022-03-21 11:20:50 - sshInstance.py - [line:50] - root - INFO - Domain VMtest2 has been undefined


2022-03-21 11:20:55 - test_network_sriov_stress.py - [line:287] - root - INFO - Successfully removed the VM: VMtest2
2022-03-21 11:20:55 - test_network_sriov_stress.py - [line:388] - root - INFO - 

********************************************************Stress Disable and Enable for 2 time when VF is Assigned VM guest.*************************************************************


2022-03-21 11:20:58 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK/dpdk-21.11/usertools/dpdk-devbind.py -s | grep enp137s0f0 | awk '{print $1;}'
2022-03-21 11:20:58 - sshInstance.py - [line:50] - root - INFO - 0000:89:00.0

2022-03-21 11:21:03 - test_network_sriov_stress.py - [line:111] - root - INFO - pf_id was found: 0000:89:00.0

2022-03-21 11:21:03 - test_network_sriov_stress.py - [line:396] - root - INFO - format_pf_id is 0000\:89\:00.0

2022-03-21 11:21:03 - test_network_sriov_stress.py - [line:399] - root - INFO - pf_id_key: 89
2022-03-21 11:21:09 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:21:09 - sshInstance.py - [line:96] - root - INFO - ifconfig enp137s0f0 192.168.10.10
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:21:12 - test_network_sriov_stress.py - [line:302] - root - INFO - Successfully assigned ip to ethernet: enp137s0f0
2022-03-21 11:21:18 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:21:19 - sshInstance.py - [line:96] - root - INFO - ifconfig ens801f0 192.168.10.11
[root@redhat8 ~]# 
2022-03-21 11:21:22 - test_network_sriov_stress.py - [line:302] - root - INFO - Successfully assigned ip to ethernet: ens801f0
2022-03-21 11:21:25 - sshInstance.py - [line:46] - root - INFO - echo 4 > /sys/class/net/enp137s0f0/device/sriov_numvfs
2022-03-21 11:21:25 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:21:30 - test_network_sriov_stress.py - [line:125] - root - INFO - Successfully Virtualized PF to VFs.
2022-03-21 11:21:33 - sshInstance.py - [line:46] - root - INFO -  lspci | grep "Virtual" | awk '{print $1;}'list
2022-03-21 11:21:33 - sshInstance.py - [line:50] - root - INFO - 89:01.0
89:01.1
89:01.2
89:01.3

2022-03-21 11:21:38 - test_network_sriov_stress.py - [line:139] - root - INFO - Succssfully Gotten vf_id list:

2022-03-21 11:21:41 - sshInstance.py - [line:46] - root - INFO - modprobe vfio-pci

2022-03-21 11:21:41 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:21:49 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK/dpdk-21.11/usertools/dpdk-devbind.py -b vfio-pci 89:01.0
2022-03-21 11:21:49 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:21:54 - test_network_sriov_stress.py - [line:172] - root - INFO - Successfuly bound VF to vfio-pci
2022-03-21 11:21:58 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK/dpdk-21.11/usertools/dpdk-devbind.py -b vfio-pci 89:01.1
2022-03-21 11:21:58 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:22:03 - test_network_sriov_stress.py - [line:172] - root - INFO - Successfuly bound VF to vfio-pci
2022-03-21 11:22:03 - test_network_sriov_stress.py - [line:421] - root - INFO - lspci -Dnn | grep 89:01.1
2022-03-21 11:22:06 - sshInstance.py - [line:46] - root - INFO - lspci -Dnn | grep 89:01.1
2022-03-21 11:22:06 - sshInstance.py - [line:50] - root - INFO - 0000:89:01.1 Ethernet controller [0200]: Intel Corporation Ethernet Adaptive Virtual Function [8086:1889]

2022-03-21 11:22:11 - test_network_sriov_stress.py - [line:433] - root - INFO - Successfully get vf device id: 8086:1889
2022-03-21 11:22:14 - sshInstance.py - [line:46] - root - INFO - virsh net-info default | grep "Active" | awk '{print $2;}'
2022-03-21 11:22:14 - sshInstance.py - [line:50] - root - INFO - yes

2022-03-21 11:22:19 - test_network_sriov_stress.py - [line:198] - root - INFO - Default network is active? yes

2022-03-21 11:22:23 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vvcpus=5 --network network=default --hostdev pci_0000_89_01_0 --boot hd --disk patth=/home/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=55900 --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:89:01.0 is in use by driver QEMU, domain VMtest1
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest1
otherwise, please restart your installation.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:22:23 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:22:26 - test_network_sriov_stress.py - [line:230] - root - ERROR - Failed to create a VM: virt-install --name VMtest1 --ram 4096 --cpu host --vvcpus=5 --network network=default --hostdev pci_0000_89_01_0 --boot hd --disk patth=/home/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=55900 --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:89:01.0 is in use by driver QEMU, domain VMtest1
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest1
otherwise, please restart your installation.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:22:26 - test_network_sriov_stress.py - [line:231] - root - ERROR - Failed to create a VM: 
2022-03-21 11:22:30 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vvcpus=5 --network network=default --hostdev pci_0000_89_01_0 --boot hd --disk patth=/home/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=55900 --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-03-21 11:22:32 - sshInstance.py - [line:71] - root - INFO - Domain creation completed.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:22:32 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:22:35 - test_network_sriov_stress.py - [line:233] - root - INFO - Successfully created a VM: VMtest1
2022-03-21 11:22:39 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vvcpus=5 --network network=default --hostdev pci_0000_89_01_1 --boot hd --disk patth=/home/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=55901 --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:89:01.1 is in use by driver QEMU, domain VMtest2
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest2
otherwise, please restart your installation.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:22:39 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:22:42 - test_network_sriov_stress.py - [line:230] - root - ERROR - Failed to create a VM: virt-install --name VMtest2 --ram 4096 --cpu host --vvcpus=5 --network network=default --hostdev pci_0000_89_01_1 --boot hd --disk patth=/home/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=55901 --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:89:01.1 is in use by driver QEMU, domain VMtest2
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest2
otherwise, please restart your installation.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:22:42 - test_network_sriov_stress.py - [line:231] - root - ERROR - Failed to create a VM: 
2022-03-21 11:22:46 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vvcpus=5 --network network=default --hostdev pci_0000_89_01_1 --boot hd --disk patth=/home/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=55901 --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-03-21 11:22:48 - sshInstance.py - [line:71] - root - INFO - Domain creation completed.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:22:48 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:22:51 - test_network_sriov_stress.py - [line:233] - root - INFO - Successfully created a VM: VMtest2
2022-03-21 11:22:54 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-03-21 11:22:54 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:22:59 - test_network_sriov_stress.py - [line:250] - root - ERROR - not found IP of VMtest1
2022-03-21 11:24:02 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-03-21 11:24:02 - sshInstance.py - [line:50] - root - INFO - 192.168.122.128/24

2022-03-21 11:24:08 - test_network_sriov_stress.py - [line:245] - root - INFO - Successfully got VMtest1 IP: 192.168.122.128/24

2022-03-21 11:24:11 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest2 | grep '192.168' | awk '{print $4;}'
2022-03-21 11:24:11 - sshInstance.py - [line:50] - root - INFO - 192.168.122.196/24

2022-03-21 11:24:16 - test_network_sriov_stress.py - [line:245] - root - INFO - Successfully got VMtest2 IP: 192.168.122.196/24

2022-03-21 11:24:16 - test_network_sriov_stress.py - [line:450] - root - INFO - ip of VM1: 192.168.122.128
2022-03-21 11:24:16 - test_network_sriov_stress.py - [line:451] - root - INFO - ip of VM1: 192.168.122.196
2022-03-21 11:24:20 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.128
PING 192.168.122.128 (192.168.122.128) 56(84) bytes of data.
64 bytes from 192.168.122.128: icmp_seq=1 ttl=64 time=0.087 ms
2022-03-21 11:24:21 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=2 ttl=64 time=0.076 ms
2022-03-21 11:24:22 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=3 ttl=64 time=0.073 ms
2022-03-21 11:24:23 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=4 ttl=64 time=0.061 ms
2022-03-21 11:24:24 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=5 ttl=64 time=0.069 ms

--- 192.168.122.128 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4075ms
rtt min/avg/max/mdev = 0.061/0.073/0.087/0.008 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:24:24 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:24:33 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:24:33 - sshInstance.py - [line:96] - root - INFO - ssh root@192.168.122.128
root@192.168.122.128's password: 
2022-03-21 11:24:42 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:24:42 - sshInstance.py - [line:96] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Sun 20 Mar 2022 11:26:29 PM EDT
  System load:  0.41               Processes:             140
  Usage of /:   37.2% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.128
  Swap usage:   0%

363 updates can be installed immediately.
221 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Sun Mar 20 23:21:25 2022 from 192.168.122.1
root@localhost:~# 
2022-03-21 11:24:51 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:24:51 - sshInstance.py - [line:96] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-03-21 11:24:57 - test_network_sriov_stress.py - [line:341] - root - INFO - successfully got vf pci id of vm1: 0000:00:05.0
2022-03-21 11:25:03 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:25:03 - sshInstance.py - [line:96] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-03-21 11:25:08 - test_network_sriov_stress.py - [line:355] - root - INFO - Successfully got vf device name of vm1: ens5
2022-03-21 11:25:14 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:25:14 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 192.168.10.12
root@localhost:~# 
2022-03-21 11:25:17 - test_network_sriov_stress.py - [line:302] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-03-21 11:25:23 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:25:23 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 down
root@localhost:~# 
2022-03-21 11:25:32 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:25:32 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 hw ether 1234567890ac
root@localhost:~# 
2022-03-21 11:25:41 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:25:41 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-03-21 11:25:50 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:25:50 - sshInstance.py - [line:96] - root - INFO - nmcli device connect ens5
Command 'nmcli' not found, but can be installed with:
apt install network-manager
root@localhost:~# 
2022-03-21 11:25:59 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:25:59 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.12  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::1034:56ff:fe78:90ac  prefixlen 64  scopeid 0x20<link>
        ether 12:34:56:78:90:ac  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 9  bytes 766 (766.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-03-21 11:26:08 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:26:08 - sshInstance.py - [line:96] - root - INFO - exit
logout
Connection to 192.168.122.128 closed.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:26:15 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.196
PING 192.168.122.196 (192.168.122.196) 56(84) bytes of data.
64 bytes from 192.168.122.196: icmp_seq=1 ttl=64 time=0.082 ms
2022-03-21 11:26:16 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.196: icmp_seq=2 ttl=64 time=0.111 ms
2022-03-21 11:26:17 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.196: icmp_seq=3 ttl=64 time=0.092 ms
2022-03-21 11:26:18 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.196: icmp_seq=4 ttl=64 time=0.115 ms
2022-03-21 11:26:19 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.196: icmp_seq=5 ttl=64 time=0.070 ms

--- 192.168.122.196 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4099ms
rtt min/avg/max/mdev = 0.070/0.094/0.115/0.017 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:26:19 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:26:28 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:26:28 - sshInstance.py - [line:96] - root - INFO - ssh root@192.168.122.196
root@192.168.122.196's password: 
2022-03-21 11:26:37 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:26:37 - sshInstance.py - [line:96] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Sun 20 Mar 2022 11:28:24 PM EDT
  System load:  0.0                Processes:             139
  Usage of /:   37.2% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.196
  Swap usage:   0%

363 updates can be installed immediately.
221 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Sun Mar 20 23:19:08 2022 from 192.168.122.1
root@localhost:~# 
2022-03-21 11:26:46 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:26:46 - sshInstance.py - [line:96] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-03-21 11:26:52 - test_network_sriov_stress.py - [line:341] - root - INFO - successfully got vf pci id of vm1: 0000:00:05.0
2022-03-21 11:26:58 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:26:58 - sshInstance.py - [line:96] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-03-21 11:27:03 - test_network_sriov_stress.py - [line:355] - root - INFO - Successfully got vf device name of vm1: ens5
2022-03-21 11:27:09 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:27:09 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 192.168.10.13
root@localhost:~# 
2022-03-21 11:27:12 - test_network_sriov_stress.py - [line:302] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-03-21 11:27:18 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:27:18 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 down
root@localhost:~# 
2022-03-21 11:27:27 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:27:27 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 hw ether 1234567890ab
root@localhost:~# 
2022-03-21 11:27:36 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:27:36 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-03-21 11:27:45 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:27:45 - sshInstance.py - [line:96] - root - INFO - nmcli device connect ens5
Command 'nmcli' not found, but can be installed with:
apt install network-manager
root@localhost:~# 
2022-03-21 11:27:54 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:27:54 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.13  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::1034:56ff:fe78:90ab  prefixlen 64  scopeid 0x20<link>
        ether 12:34:56:78:90:ab  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 9  bytes 766 (766.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-03-21 11:28:03 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:28:03 - sshInstance.py - [line:96] - root - INFO - exit
logout
Connection to 192.168.122.196 closed.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:28:06 - test_network_sriov_stress.py - [line:469] - root - INFO - Ping two vms IP from SUT:
2022-03-21 11:28:10 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.12
PING 192.168.10.12 (192.168.10.12) 56(84) bytes of data.
64 bytes from 192.168.10.12: icmp_seq=1 ttl=64 time=0.099 ms
2022-03-21 11:28:11 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=2 ttl=64 time=0.041 ms
2022-03-21 11:28:12 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=3 ttl=64 time=0.059 ms
2022-03-21 11:28:13 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=4 ttl=64 time=0.039 ms
2022-03-21 11:28:14 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=5 ttl=64 time=0.036 ms

--- 192.168.10.12 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4090ms
rtt min/avg/max/mdev = 0.036/0.054/0.099/0.023 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:28:14 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:28:21 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=0.247 ms
2022-03-21 11:28:22 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.056 ms
2022-03-21 11:28:23 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.057 ms
2022-03-21 11:28:24 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.039 ms
2022-03-21 11:28:25 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.035 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4083ms
rtt min/avg/max/mdev = 0.035/0.086/0.247/0.080 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:28:25 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:28:28 - test_network_sriov_stress.py - [line:477] - root - INFO - ping VMtest2 IP from VMtest1
2022-03-21 11:28:32 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.128
PING 192.168.122.128 (192.168.122.128) 56(84) bytes of data.
64 bytes from 192.168.122.128: icmp_seq=1 ttl=64 time=0.086 ms
2022-03-21 11:28:33 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=2 ttl=64 time=0.071 ms
2022-03-21 11:28:34 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=3 ttl=64 time=0.078 ms
2022-03-21 11:28:35 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=4 ttl=64 time=0.073 ms
2022-03-21 11:28:36 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=5 ttl=64 time=0.061 ms

--- 192.168.122.128 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4074ms
rtt min/avg/max/mdev = 0.061/0.073/0.086/0.008 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:28:36 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:28:45 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:28:45 - sshInstance.py - [line:96] - root - INFO - ssh root@192.168.122.128
root@192.168.122.128's password: 
2022-03-21 11:28:54 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:28:54 - sshInstance.py - [line:96] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Sun 20 Mar 2022 11:30:41 PM EDT
  System load:  0.0                Processes:             130
  Usage of /:   37.2% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.128
  Swap usage:   0%                 IPv4 address for ens5: 192.168.10.12

363 updates can be installed immediately.
221 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Sun Mar 20 23:26:29 2022 from 192.168.122.1
root@localhost:~# 
2022-03-21 11:29:01 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=0.141 ms
2022-03-21 11:29:02 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.035 ms
2022-03-21 11:29:03 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.026 ms
2022-03-21 11:29:04 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.037 ms
2022-03-21 11:29:05 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.033 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4077ms
rtt min/avg/max/mdev = 0.026/0.054/0.141/0.043 ms
root@localhost:~# 
2022-03-21 11:29:05 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:29:14 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:29:14 - sshInstance.py - [line:96] - root - INFO - exit
logout
Connection to 192.168.122.128 closed.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:29:17 - test_network_sriov_stress.py - [line:485] - root - INFO - ping VMtest2 IP from VMtest1
2022-03-21 11:29:21 - sshInstance.py - [line:71] - root - INFO - ping -c 5 -I 192.168.10.10 192.168.10.11
PING 192.168.10.11 (192.168.10.11) from 192.168.10.10 : 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.059 ms
2022-03-21 11:29:22 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.045 ms
2022-03-21 11:29:23 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.053 ms
2022-03-21 11:29:24 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.041 ms
2022-03-21 11:29:25 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.044 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4101ms
rtt min/avg/max/mdev = 0.041/0.048/0.059/0.006 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:29:25 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:29:31 - sshInstance.py - [line:46] - root - INFO - echo 0 > /sys/class/net/enp137s0f0/device/sriov_numvfs
2022-03-21 11:29:34 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:29:39 - test_network_sriov_stress.py - [line:264] - root - INFO - Successfully removed VFs from PF.
2022-03-21 11:29:42 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest1
2022-03-21 11:29:42 - sshInstance.py - [line:50] - root - INFO - Domain VMtest1 destroyed


2022-03-21 11:29:47 - test_network_sriov_stress.py - [line:277] - root - INFO - Successfully shut down the VM: VMtest1
2022-03-21 11:29:50 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest1
2022-03-21 11:29:50 - sshInstance.py - [line:50] - root - INFO - Domain VMtest1 has been undefined


2022-03-21 11:29:55 - test_network_sriov_stress.py - [line:287] - root - INFO - Successfully removed the VM: VMtest1
2022-03-21 11:29:58 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest2
2022-03-21 11:29:58 - sshInstance.py - [line:50] - root - INFO - Domain VMtest2 destroyed


2022-03-21 11:30:03 - test_network_sriov_stress.py - [line:277] - root - INFO - Successfully shut down the VM: VMtest2
2022-03-21 11:30:06 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest2
2022-03-21 11:30:06 - sshInstance.py - [line:50] - root - INFO - Domain VMtest2 has been undefined


2022-03-21 11:30:11 - test_network_sriov_stress.py - [line:287] - root - INFO - Successfully removed the VM: VMtest2
2022-03-21 11:30:11 - test_network_sriov_stress.py - [line:388] - root - INFO - 

********************************************************Stress Disable and Enable for 3 time when VF is Assigned VM guest.*************************************************************


2022-03-21 11:30:14 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK/dpdk-21.11/usertools/dpdk-devbind.py -s | grep enp137s0f0 | awk '{print $1;}'
2022-03-21 11:30:15 - sshInstance.py - [line:50] - root - INFO - 0000:89:00.0

2022-03-21 11:30:20 - test_network_sriov_stress.py - [line:111] - root - INFO - pf_id was found: 0000:89:00.0

2022-03-21 11:30:20 - test_network_sriov_stress.py - [line:396] - root - INFO - format_pf_id is 0000\:89\:00.0

2022-03-21 11:30:20 - test_network_sriov_stress.py - [line:399] - root - INFO - pf_id_key: 89
2022-03-21 11:30:26 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:30:26 - sshInstance.py - [line:96] - root - INFO - ifconfig enp137s0f0 192.168.10.10
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:30:29 - test_network_sriov_stress.py - [line:302] - root - INFO - Successfully assigned ip to ethernet: enp137s0f0
2022-03-21 11:30:35 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:30:35 - sshInstance.py - [line:96] - root - INFO - ifconfig ens801f0 192.168.10.11
[root@redhat8 ~]# 
2022-03-21 11:30:38 - test_network_sriov_stress.py - [line:302] - root - INFO - Successfully assigned ip to ethernet: ens801f0
2022-03-21 11:30:41 - sshInstance.py - [line:46] - root - INFO - echo 4 > /sys/class/net/enp137s0f0/device/sriov_numvfs
2022-03-21 11:30:41 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:30:46 - test_network_sriov_stress.py - [line:125] - root - INFO - Successfully Virtualized PF to VFs.
2022-03-21 11:30:49 - sshInstance.py - [line:46] - root - INFO -  lspci | grep "Virtual" | awk '{print $1;}'list
2022-03-21 11:30:49 - sshInstance.py - [line:50] - root - INFO - 89:01.0
89:01.1
89:01.2
89:01.3

2022-03-21 11:30:54 - test_network_sriov_stress.py - [line:139] - root - INFO - Succssfully Gotten vf_id list:

2022-03-21 11:30:57 - sshInstance.py - [line:46] - root - INFO - modprobe vfio-pci

2022-03-21 11:30:57 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:31:05 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK/dpdk-21.11/usertools/dpdk-devbind.py -b vfio-pci 89:01.0
2022-03-21 11:31:06 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:31:11 - test_network_sriov_stress.py - [line:172] - root - INFO - Successfuly bound VF to vfio-pci
2022-03-21 11:31:14 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK/dpdk-21.11/usertools/dpdk-devbind.py -b vfio-pci 89:01.1
2022-03-21 11:31:15 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:31:20 - test_network_sriov_stress.py - [line:172] - root - INFO - Successfuly bound VF to vfio-pci
2022-03-21 11:31:20 - test_network_sriov_stress.py - [line:421] - root - INFO - lspci -Dnn | grep 89:01.1
2022-03-21 11:31:23 - sshInstance.py - [line:46] - root - INFO - lspci -Dnn | grep 89:01.1
2022-03-21 11:31:23 - sshInstance.py - [line:50] - root - INFO - 0000:89:01.1 Ethernet controller [0200]: Intel Corporation Ethernet Adaptive Virtual Function [8086:1889]

2022-03-21 11:31:28 - test_network_sriov_stress.py - [line:433] - root - INFO - Successfully get vf device id: 8086:1889
2022-03-21 11:31:31 - sshInstance.py - [line:46] - root - INFO - virsh net-info default | grep "Active" | awk '{print $2;}'
2022-03-21 11:31:31 - sshInstance.py - [line:50] - root - INFO - yes

2022-03-21 11:31:36 - test_network_sriov_stress.py - [line:198] - root - INFO - Default network is active? yes

2022-03-21 11:31:40 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vvcpus=5 --network network=default --hostdev pci_0000_89_01_0 --boot hd --disk patth=/home/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=55900 --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:89:01.0 is in use by driver QEMU, domain VMtest1
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest1
otherwise, please restart your installation.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:31:40 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:31:43 - test_network_sriov_stress.py - [line:230] - root - ERROR - Failed to create a VM: virt-install --name VMtest1 --ram 4096 --cpu host --vvcpus=5 --network network=default --hostdev pci_0000_89_01_0 --boot hd --disk patth=/home/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=55900 --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:89:01.0 is in use by driver QEMU, domain VMtest1
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest1
otherwise, please restart your installation.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:31:43 - test_network_sriov_stress.py - [line:231] - root - ERROR - Failed to create a VM: 
2022-03-21 11:31:47 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vvcpus=5 --network network=default --hostdev pci_0000_89_01_0 --boot hd --disk patth=/home/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=55900 --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-03-21 11:31:49 - sshInstance.py - [line:71] - root - INFO - Domain creation completed.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:31:49 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:31:52 - test_network_sriov_stress.py - [line:233] - root - INFO - Successfully created a VM: VMtest1
2022-03-21 11:31:56 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vvcpus=5 --network network=default --hostdev pci_0000_89_01_1 --boot hd --disk patth=/home/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=55901 --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:89:01.1 is in use by driver QEMU, domain VMtest2
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest2
otherwise, please restart your installation.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:31:56 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:31:59 - test_network_sriov_stress.py - [line:230] - root - ERROR - Failed to create a VM: virt-install --name VMtest2 --ram 4096 --cpu host --vvcpus=5 --network network=default --hostdev pci_0000_89_01_1 --boot hd --disk patth=/home/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=55901 --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:89:01.1 is in use by driver QEMU, domain VMtest2
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest2
otherwise, please restart your installation.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:31:59 - test_network_sriov_stress.py - [line:231] - root - ERROR - Failed to create a VM: 
2022-03-21 11:32:03 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vvcpus=5 --network network=default --hostdev pci_0000_89_01_1 --boot hd --disk patth=/home/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=55901 --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-03-21 11:32:05 - sshInstance.py - [line:71] - root - INFO - Domain creation completed.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:32:05 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:32:08 - test_network_sriov_stress.py - [line:233] - root - INFO - Successfully created a VM: VMtest2
2022-03-21 11:32:11 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-03-21 11:32:11 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:32:16 - test_network_sriov_stress.py - [line:250] - root - ERROR - not found IP of VMtest1
2022-03-21 11:33:19 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-03-21 11:33:19 - sshInstance.py - [line:50] - root - INFO - 192.168.122.128/24

2022-03-21 11:33:24 - test_network_sriov_stress.py - [line:245] - root - INFO - Successfully got VMtest1 IP: 192.168.122.128/24

2022-03-21 11:33:27 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest2 | grep '192.168' | awk '{print $4;}'
2022-03-21 11:33:27 - sshInstance.py - [line:50] - root - INFO - 192.168.122.196/24

2022-03-21 11:33:32 - test_network_sriov_stress.py - [line:245] - root - INFO - Successfully got VMtest2 IP: 192.168.122.196/24

2022-03-21 11:33:32 - test_network_sriov_stress.py - [line:450] - root - INFO - ip of VM1: 192.168.122.128
2022-03-21 11:33:32 - test_network_sriov_stress.py - [line:451] - root - INFO - ip of VM1: 192.168.122.196
2022-03-21 11:33:36 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.128
PING 192.168.122.128 (192.168.122.128) 56(84) bytes of data.
64 bytes from 192.168.122.128: icmp_seq=1 ttl=64 time=0.119 ms
2022-03-21 11:33:37 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=2 ttl=64 time=0.070 ms
2022-03-21 11:33:38 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=3 ttl=64 time=0.071 ms
2022-03-21 11:33:39 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=4 ttl=64 time=0.068 ms
2022-03-21 11:33:40 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=5 ttl=64 time=0.068 ms

--- 192.168.122.128 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4100ms
rtt min/avg/max/mdev = 0.068/0.079/0.119/0.019 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:33:40 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:33:49 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:33:49 - sshInstance.py - [line:96] - root - INFO - ssh root@192.168.122.128
root@192.168.122.128's password: 
2022-03-21 11:33:58 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:33:58 - sshInstance.py - [line:96] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Sun 20 Mar 2022 11:35:46 PM EDT
  System load:  0.0                Processes:             138
  Usage of /:   37.2% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.128
  Swap usage:   0%

363 updates can be installed immediately.
221 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Sun Mar 20 23:30:41 2022 from 192.168.122.1
root@localhost:~# 
2022-03-21 11:34:07 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:34:07 - sshInstance.py - [line:96] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-03-21 11:34:13 - test_network_sriov_stress.py - [line:341] - root - INFO - successfully got vf pci id of vm1: 0000:00:05.0
2022-03-21 11:34:19 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:34:19 - sshInstance.py - [line:96] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-03-21 11:34:24 - test_network_sriov_stress.py - [line:355] - root - INFO - Successfully got vf device name of vm1: ens5
2022-03-21 11:34:30 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:34:30 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 192.168.10.12
root@localhost:~# 
2022-03-21 11:34:33 - test_network_sriov_stress.py - [line:302] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-03-21 11:34:39 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:34:39 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 down
root@localhost:~# 
2022-03-21 11:34:48 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:34:48 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 hw ether 1234567890ac
root@localhost:~# 
2022-03-21 11:34:57 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:34:57 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-03-21 11:35:06 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:35:06 - sshInstance.py - [line:96] - root - INFO - nmcli device connect ens5
Command 'nmcli' not found, but can be installed with:
apt install network-manager
root@localhost:~# 
2022-03-21 11:35:15 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:35:15 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.12  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::1034:56ff:fe78:90ac  prefixlen 64  scopeid 0x20<link>
        ether 12:34:56:78:90:ac  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 9  bytes 766 (766.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-03-21 11:35:24 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:35:24 - sshInstance.py - [line:96] - root - INFO - exit
logout
Connection to 192.168.122.128 closed.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:35:31 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.196
PING 192.168.122.196 (192.168.122.196) 56(84) bytes of data.
64 bytes from 192.168.122.196: icmp_seq=1 ttl=64 time=0.087 ms
2022-03-21 11:35:32 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.196: icmp_seq=2 ttl=64 time=0.074 ms
2022-03-21 11:35:33 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.196: icmp_seq=3 ttl=64 time=0.083 ms
2022-03-21 11:35:34 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.196: icmp_seq=4 ttl=64 time=0.091 ms
2022-03-21 11:35:35 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.196: icmp_seq=5 ttl=64 time=0.081 ms

--- 192.168.122.196 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4095ms
rtt min/avg/max/mdev = 0.074/0.083/0.091/0.005 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:35:35 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:35:44 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:35:44 - sshInstance.py - [line:96] - root - INFO - ssh root@192.168.122.196
root@192.168.122.196's password: 
2022-03-21 11:35:53 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:35:53 - sshInstance.py - [line:96] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Sun 20 Mar 2022 11:37:40 PM EDT
  System load:  0.01               Processes:             138
  Usage of /:   37.3% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.196
  Swap usage:   0%

363 updates can be installed immediately.
221 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Sun Mar 20 23:28:24 2022 from 192.168.122.1
root@localhost:~# 
2022-03-21 11:36:02 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:36:02 - sshInstance.py - [line:96] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-03-21 11:36:08 - test_network_sriov_stress.py - [line:341] - root - INFO - successfully got vf pci id of vm1: 0000:00:05.0
2022-03-21 11:36:14 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:36:14 - sshInstance.py - [line:96] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-03-21 11:36:19 - test_network_sriov_stress.py - [line:355] - root - INFO - Successfully got vf device name of vm1: ens5
2022-03-21 11:36:25 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:36:25 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 192.168.10.13
root@localhost:~# 
2022-03-21 11:36:28 - test_network_sriov_stress.py - [line:302] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-03-21 11:36:34 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:36:34 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 down
root@localhost:~# 
2022-03-21 11:36:43 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:36:43 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 hw ether 1234567890ab
root@localhost:~# 
2022-03-21 11:36:52 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:36:52 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-03-21 11:37:01 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:37:01 - sshInstance.py - [line:96] - root - INFO - nmcli device connect ens5
Command 'nmcli' not found, but can be installed with:
apt install network-manager
root@localhost:~# 
2022-03-21 11:37:10 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:37:10 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.13  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::1034:56ff:fe78:90ab  prefixlen 64  scopeid 0x20<link>
        ether 12:34:56:78:90:ab  txqueuelen 1000  (Ethernet)
        RX packets 1  bytes 345 (345.0 B)
        RX errors 0  dropped 1  overruns 0  frame 0
        TX packets 9  bytes 766 (766.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-03-21 11:37:19 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:37:19 - sshInstance.py - [line:96] - root - INFO - exit
logout
Connection to 192.168.122.196 closed.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:37:22 - test_network_sriov_stress.py - [line:469] - root - INFO - Ping two vms IP from SUT:
2022-03-21 11:37:26 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.12
PING 192.168.10.12 (192.168.10.12) 56(84) bytes of data.
64 bytes from 192.168.10.12: icmp_seq=1 ttl=64 time=0.320 ms
2022-03-21 11:37:27 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=2 ttl=64 time=0.043 ms
2022-03-21 11:37:28 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=3 ttl=64 time=0.064 ms
2022-03-21 11:37:29 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=4 ttl=64 time=0.045 ms
2022-03-21 11:37:30 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=5 ttl=64 time=0.055 ms

--- 192.168.10.12 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4086ms
rtt min/avg/max/mdev = 0.043/0.105/0.320/0.107 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:37:30 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:37:37 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=0.122 ms
2022-03-21 11:37:38 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.034 ms
2022-03-21 11:37:39 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.022 ms
2022-03-21 11:37:40 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.043 ms
2022-03-21 11:37:41 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.036 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4079ms
rtt min/avg/max/mdev = 0.022/0.051/0.122/0.035 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:37:41 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:37:44 - test_network_sriov_stress.py - [line:477] - root - INFO - ping VMtest2 IP from VMtest1
2022-03-21 11:37:48 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.128
PING 192.168.122.128 (192.168.122.128) 56(84) bytes of data.
64 bytes from 192.168.122.128: icmp_seq=1 ttl=64 time=0.090 ms
2022-03-21 11:37:49 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=2 ttl=64 time=0.083 ms
2022-03-21 11:37:50 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=3 ttl=64 time=0.080 ms
2022-03-21 11:37:51 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=4 ttl=64 time=0.084 ms
2022-03-21 11:37:52 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=5 ttl=64 time=0.076 ms

--- 192.168.122.128 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4102ms
rtt min/avg/max/mdev = 0.076/0.082/0.090/0.004 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:37:52 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:38:01 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:38:01 - sshInstance.py - [line:96] - root - INFO - ssh root@192.168.122.128
root@192.168.122.128's password: 
2022-03-21 11:38:10 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:38:10 - sshInstance.py - [line:96] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Sun 20 Mar 2022 11:39:58 PM EDT
  System load:  0.0                Processes:             126
  Usage of /:   37.2% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.128
  Swap usage:   0%                 IPv4 address for ens5: 192.168.10.12

363 updates can be installed immediately.
221 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Sun Mar 20 23:35:46 2022 from 192.168.122.1
root@localhost:~# 
2022-03-21 11:38:17 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=0.121 ms
2022-03-21 11:38:18 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.035 ms
2022-03-21 11:38:19 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.029 ms
2022-03-21 11:38:20 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.030 ms
2022-03-21 11:38:21 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.031 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4084ms
rtt min/avg/max/mdev = 0.029/0.049/0.121/0.035 ms
root@localhost:~# 
2022-03-21 11:38:21 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:38:30 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:38:30 - sshInstance.py - [line:96] - root - INFO - exit
logout
Connection to 192.168.122.128 closed.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:38:33 - test_network_sriov_stress.py - [line:485] - root - INFO - ping VMtest2 IP from VMtest1
2022-03-21 11:38:37 - sshInstance.py - [line:71] - root - INFO - ping -c 5 -I 192.168.10.10 192.168.10.11
PING 192.168.10.11 (192.168.10.11) from 192.168.10.10 : 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.059 ms
2022-03-21 11:38:38 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.041 ms
2022-03-21 11:38:39 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.037 ms
2022-03-21 11:38:40 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.034 ms
2022-03-21 11:38:41 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.033 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4096ms
rtt min/avg/max/mdev = 0.033/0.040/0.059/0.009 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:38:41 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:38:47 - sshInstance.py - [line:46] - root - INFO - echo 0 > /sys/class/net/enp137s0f0/device/sriov_numvfs
2022-03-21 11:38:50 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:38:55 - test_network_sriov_stress.py - [line:264] - root - INFO - Successfully removed VFs from PF.
2022-03-21 11:38:58 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest1
2022-03-21 11:38:58 - sshInstance.py - [line:50] - root - INFO - Domain VMtest1 destroyed


2022-03-21 11:39:03 - test_network_sriov_stress.py - [line:277] - root - INFO - Successfully shut down the VM: VMtest1
2022-03-21 11:39:06 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest1
2022-03-21 11:39:06 - sshInstance.py - [line:50] - root - INFO - Domain VMtest1 has been undefined


2022-03-21 11:39:11 - test_network_sriov_stress.py - [line:287] - root - INFO - Successfully removed the VM: VMtest1
2022-03-21 11:39:14 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest2
2022-03-21 11:39:15 - sshInstance.py - [line:50] - root - INFO - Domain VMtest2 destroyed


2022-03-21 11:39:20 - test_network_sriov_stress.py - [line:277] - root - INFO - Successfully shut down the VM: VMtest2
2022-03-21 11:39:23 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest2
2022-03-21 11:39:23 - sshInstance.py - [line:50] - root - INFO - Domain VMtest2 has been undefined


2022-03-21 11:39:28 - test_network_sriov_stress.py - [line:287] - root - INFO - Successfully removed the VM: VMtest2
2022-03-21 11:39:28 - test_network_sriov_stress.py - [line:388] - root - INFO - 

********************************************************Stress Disable and Enable for 4 time when VF is Assigned VM guest.*************************************************************


2022-03-21 11:39:31 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK/dpdk-21.11/usertools/dpdk-devbind.py -s | grep enp137s0f0 | awk '{print $1;}'
2022-03-21 11:39:31 - sshInstance.py - [line:50] - root - INFO - 0000:89:00.0

2022-03-21 11:39:36 - test_network_sriov_stress.py - [line:111] - root - INFO - pf_id was found: 0000:89:00.0

2022-03-21 11:39:36 - test_network_sriov_stress.py - [line:396] - root - INFO - format_pf_id is 0000\:89\:00.0

2022-03-21 11:39:36 - test_network_sriov_stress.py - [line:399] - root - INFO - pf_id_key: 89
2022-03-21 11:39:42 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:39:42 - sshInstance.py - [line:96] - root - INFO - ifconfig enp137s0f0 192.168.10.10
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:39:45 - test_network_sriov_stress.py - [line:302] - root - INFO - Successfully assigned ip to ethernet: enp137s0f0
2022-03-21 11:39:51 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:39:51 - sshInstance.py - [line:96] - root - INFO - ifconfig ens801f0 192.168.10.11
[root@redhat8 ~]# 
2022-03-21 11:39:54 - test_network_sriov_stress.py - [line:302] - root - INFO - Successfully assigned ip to ethernet: ens801f0
2022-03-21 11:39:57 - sshInstance.py - [line:46] - root - INFO - echo 4 > /sys/class/net/enp137s0f0/device/sriov_numvfs
2022-03-21 11:39:57 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:40:02 - test_network_sriov_stress.py - [line:125] - root - INFO - Successfully Virtualized PF to VFs.
2022-03-21 11:40:05 - sshInstance.py - [line:46] - root - INFO -  lspci | grep "Virtual" | awk '{print $1;}'list
2022-03-21 11:40:05 - sshInstance.py - [line:50] - root - INFO - 89:01.0
89:01.1
89:01.2
89:01.3

2022-03-21 11:40:10 - test_network_sriov_stress.py - [line:139] - root - INFO - Succssfully Gotten vf_id list:

2022-03-21 11:40:13 - sshInstance.py - [line:46] - root - INFO - modprobe vfio-pci

2022-03-21 11:40:13 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:40:21 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK/dpdk-21.11/usertools/dpdk-devbind.py -b vfio-pci 89:01.0
2022-03-21 11:40:22 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:40:27 - test_network_sriov_stress.py - [line:172] - root - INFO - Successfuly bound VF to vfio-pci
2022-03-21 11:40:30 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK/dpdk-21.11/usertools/dpdk-devbind.py -b vfio-pci 89:01.1
2022-03-21 11:40:31 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:40:36 - test_network_sriov_stress.py - [line:172] - root - INFO - Successfuly bound VF to vfio-pci
2022-03-21 11:40:36 - test_network_sriov_stress.py - [line:421] - root - INFO - lspci -Dnn | grep 89:01.1
2022-03-21 11:40:39 - sshInstance.py - [line:46] - root - INFO - lspci -Dnn | grep 89:01.1
2022-03-21 11:40:39 - sshInstance.py - [line:50] - root - INFO - 0000:89:01.1 Ethernet controller [0200]: Intel Corporation Ethernet Adaptive Virtual Function [8086:1889]

2022-03-21 11:40:44 - test_network_sriov_stress.py - [line:433] - root - INFO - Successfully get vf device id: 8086:1889
2022-03-21 11:40:47 - sshInstance.py - [line:46] - root - INFO - virsh net-info default | grep "Active" | awk '{print $2;}'
2022-03-21 11:40:47 - sshInstance.py - [line:50] - root - INFO - yes

2022-03-21 11:40:52 - test_network_sriov_stress.py - [line:198] - root - INFO - Default network is active? yes

2022-03-21 11:40:56 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vvcpus=5 --network network=default --hostdev pci_0000_89_01_0 --boot hd --disk patth=/home/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=55900 --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:89:01.0 is in use by driver QEMU, domain VMtest1
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest1
otherwise, please restart your installation.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:40:56 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:40:59 - test_network_sriov_stress.py - [line:230] - root - ERROR - Failed to create a VM: virt-install --name VMtest1 --ram 4096 --cpu host --vvcpus=5 --network network=default --hostdev pci_0000_89_01_0 --boot hd --disk patth=/home/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=55900 --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:89:01.0 is in use by driver QEMU, domain VMtest1
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest1
otherwise, please restart your installation.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:40:59 - test_network_sriov_stress.py - [line:231] - root - ERROR - Failed to create a VM: 
2022-03-21 11:41:03 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vvcpus=5 --network network=default --hostdev pci_0000_89_01_0 --boot hd --disk patth=/home/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=55900 --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-03-21 11:41:05 - sshInstance.py - [line:71] - root - INFO - Domain creation completed.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:41:05 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:41:08 - test_network_sriov_stress.py - [line:233] - root - INFO - Successfully created a VM: VMtest1
2022-03-21 11:41:12 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vvcpus=5 --network network=default --hostdev pci_0000_89_01_1 --boot hd --disk patth=/home/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=55901 --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:89:01.1 is in use by driver QEMU, domain VMtest2
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest2
otherwise, please restart your installation.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:41:12 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:41:15 - test_network_sriov_stress.py - [line:230] - root - ERROR - Failed to create a VM: virt-install --name VMtest2 --ram 4096 --cpu host --vvcpus=5 --network network=default --hostdev pci_0000_89_01_1 --boot hd --disk patth=/home/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=55901 --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:89:01.1 is in use by driver QEMU, domain VMtest2
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest2
otherwise, please restart your installation.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:41:15 - test_network_sriov_stress.py - [line:231] - root - ERROR - Failed to create a VM: 
2022-03-21 11:41:19 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vvcpus=5 --network network=default --hostdev pci_0000_89_01_1 --boot hd --disk patth=/home/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=55901 --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-03-21 11:41:21 - sshInstance.py - [line:71] - root - INFO - Domain creation completed.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:41:21 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:41:24 - test_network_sriov_stress.py - [line:233] - root - INFO - Successfully created a VM: VMtest2
2022-03-21 11:41:27 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-03-21 11:41:27 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:41:32 - test_network_sriov_stress.py - [line:250] - root - ERROR - not found IP of VMtest1
2022-03-21 11:42:35 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-03-21 11:42:35 - sshInstance.py - [line:50] - root - INFO - 192.168.122.128/24

2022-03-21 11:42:40 - test_network_sriov_stress.py - [line:245] - root - INFO - Successfully got VMtest1 IP: 192.168.122.128/24

2022-03-21 11:42:43 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest2 | grep '192.168' | awk '{print $4;}'
2022-03-21 11:42:43 - sshInstance.py - [line:50] - root - INFO - 192.168.122.196/24

2022-03-21 11:42:48 - test_network_sriov_stress.py - [line:245] - root - INFO - Successfully got VMtest2 IP: 192.168.122.196/24

2022-03-21 11:42:48 - test_network_sriov_stress.py - [line:450] - root - INFO - ip of VM1: 192.168.122.128
2022-03-21 11:42:48 - test_network_sriov_stress.py - [line:451] - root - INFO - ip of VM1: 192.168.122.196
2022-03-21 11:42:52 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.128
PING 192.168.122.128 (192.168.122.128) 56(84) bytes of data.
64 bytes from 192.168.122.128: icmp_seq=1 ttl=64 time=0.088 ms
2022-03-21 11:42:53 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=2 ttl=64 time=0.072 ms
2022-03-21 11:42:54 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=3 ttl=64 time=0.072 ms
2022-03-21 11:42:55 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=4 ttl=64 time=0.071 ms
2022-03-21 11:42:56 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=5 ttl=64 time=0.069 ms

--- 192.168.122.128 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4077ms
rtt min/avg/max/mdev = 0.069/0.074/0.088/0.006 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:42:56 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:43:05 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:43:05 - sshInstance.py - [line:96] - root - INFO - ssh root@192.168.122.128
root@192.168.122.128's password: 
2022-03-21 11:43:14 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:43:14 - sshInstance.py - [line:96] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Sun 20 Mar 2022 11:45:02 PM EDT
  System load:  0.0                Processes:             136
  Usage of /:   37.3% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.128
  Swap usage:   0%

363 updates can be installed immediately.
221 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Sun Mar 20 23:39:58 2022 from 192.168.122.1
root@localhost:~# 
2022-03-21 11:43:23 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:43:23 - sshInstance.py - [line:96] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-03-21 11:43:29 - test_network_sriov_stress.py - [line:341] - root - INFO - successfully got vf pci id of vm1: 0000:00:05.0
2022-03-21 11:43:35 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:43:35 - sshInstance.py - [line:96] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-03-21 11:43:40 - test_network_sriov_stress.py - [line:355] - root - INFO - Successfully got vf device name of vm1: ens5
2022-03-21 11:43:46 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:43:46 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 192.168.10.12
root@localhost:~# 
2022-03-21 11:43:49 - test_network_sriov_stress.py - [line:302] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-03-21 11:43:55 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:43:55 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 down
root@localhost:~# 
2022-03-21 11:44:04 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:44:04 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 hw ether 1234567890ac
root@localhost:~# 
2022-03-21 11:44:13 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:44:13 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-03-21 11:44:22 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:44:22 - sshInstance.py - [line:96] - root - INFO - nmcli device connect ens5
Command 'nmcli' not found, but can be installed with:
apt install network-manager
root@localhost:~# 
2022-03-21 11:44:31 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:44:31 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.12  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::1034:56ff:fe78:90ac  prefixlen 64  scopeid 0x20<link>
        ether 12:34:56:78:90:ac  txqueuelen 1000  (Ethernet)
        RX packets 1  bytes 345 (345.0 B)
        RX errors 0  dropped 1  overruns 0  frame 0
        TX packets 9  bytes 766 (766.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-03-21 11:44:40 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:44:40 - sshInstance.py - [line:96] - root - INFO - exit
logout
Connection to 192.168.122.128 closed.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:44:47 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.196
PING 192.168.122.196 (192.168.122.196) 56(84) bytes of data.
64 bytes from 192.168.122.196: icmp_seq=1 ttl=64 time=0.093 ms
2022-03-21 11:44:48 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.196: icmp_seq=2 ttl=64 time=0.071 ms
2022-03-21 11:44:49 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.196: icmp_seq=3 ttl=64 time=0.072 ms
2022-03-21 11:44:50 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.196: icmp_seq=4 ttl=64 time=0.075 ms
2022-03-21 11:44:51 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.196: icmp_seq=5 ttl=64 time=0.063 ms

--- 192.168.122.196 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4098ms
rtt min/avg/max/mdev = 0.063/0.074/0.093/0.009 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:44:51 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:45:00 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:45:00 - sshInstance.py - [line:96] - root - INFO - ssh root@192.168.122.196
root@192.168.122.196's password: 
2022-03-21 11:45:09 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:45:09 - sshInstance.py - [line:96] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Sun 20 Mar 2022 11:46:56 PM EDT
  System load:  0.0                Processes:             139
  Usage of /:   37.3% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.196
  Swap usage:   0%

363 updates can be installed immediately.
221 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Sun Mar 20 23:37:41 2022 from 192.168.122.1
root@localhost:~# 
2022-03-21 11:45:18 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:45:18 - sshInstance.py - [line:96] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-03-21 11:45:24 - test_network_sriov_stress.py - [line:341] - root - INFO - successfully got vf pci id of vm1: 0000:00:05.0
2022-03-21 11:45:30 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:45:30 - sshInstance.py - [line:96] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-03-21 11:45:35 - test_network_sriov_stress.py - [line:355] - root - INFO - Successfully got vf device name of vm1: ens5
2022-03-21 11:45:41 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:45:41 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 192.168.10.13
root@localhost:~# 
2022-03-21 11:45:44 - test_network_sriov_stress.py - [line:302] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-03-21 11:45:50 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:45:50 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 down
root@localhost:~# 
2022-03-21 11:45:59 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:45:59 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 hw ether 1234567890ab
root@localhost:~# 
2022-03-21 11:46:08 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:46:08 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-03-21 11:46:17 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:46:17 - sshInstance.py - [line:96] - root - INFO - nmcli device connect ens5
Command 'nmcli' not found, but can be installed with:
apt install network-manager
root@localhost:~# 
2022-03-21 11:46:26 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:46:26 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.13  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::1034:56ff:fe78:90ab  prefixlen 64  scopeid 0x20<link>
        ether 12:34:56:78:90:ab  txqueuelen 1000  (Ethernet)
        RX packets 1  bytes 345 (345.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 9  bytes 766 (766.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-03-21 11:46:35 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:46:35 - sshInstance.py - [line:96] - root - INFO - exit
logout
Connection to 192.168.122.196 closed.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:46:38 - test_network_sriov_stress.py - [line:469] - root - INFO - Ping two vms IP from SUT:
2022-03-21 11:46:42 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.12
PING 192.168.10.12 (192.168.10.12) 56(84) bytes of data.
64 bytes from 192.168.10.12: icmp_seq=1 ttl=64 time=0.285 ms
2022-03-21 11:46:43 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=2 ttl=64 time=0.052 ms
2022-03-21 11:46:44 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=3 ttl=64 time=0.051 ms
2022-03-21 11:46:45 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=4 ttl=64 time=0.028 ms
2022-03-21 11:46:47 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=5 ttl=64 time=0.039 ms

--- 192.168.10.12 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4088ms
rtt min/avg/max/mdev = 0.028/0.091/0.285/0.097 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:46:47 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:46:54 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=0.255 ms
2022-03-21 11:46:55 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.047 ms
2022-03-21 11:46:56 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.028 ms
2022-03-21 11:46:57 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.034 ms
2022-03-21 11:46:58 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.030 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4081ms
rtt min/avg/max/mdev = 0.028/0.078/0.255/0.088 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:46:58 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:47:01 - test_network_sriov_stress.py - [line:477] - root - INFO - ping VMtest2 IP from VMtest1
2022-03-21 11:47:05 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.128
PING 192.168.122.128 (192.168.122.128) 56(84) bytes of data.
64 bytes from 192.168.122.128: icmp_seq=1 ttl=64 time=0.090 ms
2022-03-21 11:47:06 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=2 ttl=64 time=0.075 ms
2022-03-21 11:47:07 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=3 ttl=64 time=0.055 ms
2022-03-21 11:47:08 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=4 ttl=64 time=0.067 ms
2022-03-21 11:47:09 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=5 ttl=64 time=0.069 ms

--- 192.168.122.128 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4072ms
rtt min/avg/max/mdev = 0.055/0.071/0.090/0.011 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:47:09 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:47:18 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:47:18 - sshInstance.py - [line:96] - root - INFO - ssh root@192.168.122.128
root@192.168.122.128's password: 
2022-03-21 11:47:27 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:47:27 - sshInstance.py - [line:96] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Sun 20 Mar 2022 11:49:14 PM EDT
  System load:  0.0                Processes:             131
  Usage of /:   37.3% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.128
  Swap usage:   0%                 IPv4 address for ens5: 192.168.10.12

363 updates can be installed immediately.
221 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Sun Mar 20 23:45:02 2022 from 192.168.122.1
root@localhost:~# 
2022-03-21 11:47:34 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=0.132 ms
2022-03-21 11:47:35 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.036 ms
2022-03-21 11:47:36 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.031 ms
2022-03-21 11:47:37 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.031 ms
2022-03-21 11:47:38 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.038 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4104ms
rtt min/avg/max/mdev = 0.031/0.053/0.132/0.039 ms
root@localhost:~# 
2022-03-21 11:47:38 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:47:47 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:47:47 - sshInstance.py - [line:96] - root - INFO - exit
logout
Connection to 192.168.122.128 closed.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:47:50 - test_network_sriov_stress.py - [line:485] - root - INFO - ping VMtest2 IP from VMtest1
2022-03-21 11:47:54 - sshInstance.py - [line:71] - root - INFO - ping -c 5 -I 192.168.10.10 192.168.10.11
PING 192.168.10.11 (192.168.10.11) from 192.168.10.10 : 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.068 ms
2022-03-21 11:47:55 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.040 ms
2022-03-21 11:47:56 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.037 ms
2022-03-21 11:47:57 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.036 ms
2022-03-21 11:47:58 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.033 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4097ms
rtt min/avg/max/mdev = 0.033/0.042/0.068/0.012 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:47:58 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:48:04 - sshInstance.py - [line:46] - root - INFO - echo 0 > /sys/class/net/enp137s0f0/device/sriov_numvfs
2022-03-21 11:48:06 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:48:11 - test_network_sriov_stress.py - [line:264] - root - INFO - Successfully removed VFs from PF.
2022-03-21 11:48:14 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest1
2022-03-21 11:48:15 - sshInstance.py - [line:50] - root - INFO - Domain VMtest1 destroyed


2022-03-21 11:48:20 - test_network_sriov_stress.py - [line:277] - root - INFO - Successfully shut down the VM: VMtest1
2022-03-21 11:48:23 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest1
2022-03-21 11:48:23 - sshInstance.py - [line:50] - root - INFO - Domain VMtest1 has been undefined


2022-03-21 11:48:28 - test_network_sriov_stress.py - [line:287] - root - INFO - Successfully removed the VM: VMtest1
2022-03-21 11:48:31 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest2
2022-03-21 11:48:31 - sshInstance.py - [line:50] - root - INFO - Domain VMtest2 destroyed


2022-03-21 11:48:36 - test_network_sriov_stress.py - [line:277] - root - INFO - Successfully shut down the VM: VMtest2
2022-03-21 11:48:39 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest2
2022-03-21 11:48:39 - sshInstance.py - [line:50] - root - INFO - Domain VMtest2 has been undefined


2022-03-21 11:48:44 - test_network_sriov_stress.py - [line:287] - root - INFO - Successfully removed the VM: VMtest2
2022-03-21 11:48:44 - test_network_sriov_stress.py - [line:388] - root - INFO - 

********************************************************Stress Disable and Enable for 5 time when VF is Assigned VM guest.*************************************************************


2022-03-21 11:48:47 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK/dpdk-21.11/usertools/dpdk-devbind.py -s | grep enp137s0f0 | awk '{print $1;}'
2022-03-21 11:48:47 - sshInstance.py - [line:50] - root - INFO - 0000:89:00.0

2022-03-21 11:48:52 - test_network_sriov_stress.py - [line:111] - root - INFO - pf_id was found: 0000:89:00.0

2022-03-21 11:48:52 - test_network_sriov_stress.py - [line:396] - root - INFO - format_pf_id is 0000\:89\:00.0

2022-03-21 11:48:52 - test_network_sriov_stress.py - [line:399] - root - INFO - pf_id_key: 89
2022-03-21 11:48:59 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:48:59 - sshInstance.py - [line:96] - root - INFO - ifconfig enp137s0f0 192.168.10.10
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:49:02 - test_network_sriov_stress.py - [line:302] - root - INFO - Successfully assigned ip to ethernet: enp137s0f0
2022-03-21 11:49:08 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:49:08 - sshInstance.py - [line:96] - root - INFO - ifconfig ens801f0 192.168.10.11
[root@redhat8 ~]# 
2022-03-21 11:49:11 - test_network_sriov_stress.py - [line:302] - root - INFO - Successfully assigned ip to ethernet: ens801f0
2022-03-21 11:49:14 - sshInstance.py - [line:46] - root - INFO - echo 4 > /sys/class/net/enp137s0f0/device/sriov_numvfs
2022-03-21 11:49:14 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:49:19 - test_network_sriov_stress.py - [line:125] - root - INFO - Successfully Virtualized PF to VFs.
2022-03-21 11:49:22 - sshInstance.py - [line:46] - root - INFO -  lspci | grep "Virtual" | awk '{print $1;}'list
2022-03-21 11:49:22 - sshInstance.py - [line:50] - root - INFO - 89:01.0
89:01.1
89:01.2
89:01.3

2022-03-21 11:49:27 - test_network_sriov_stress.py - [line:139] - root - INFO - Succssfully Gotten vf_id list:

2022-03-21 11:49:30 - sshInstance.py - [line:46] - root - INFO - modprobe vfio-pci

2022-03-21 11:49:30 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:49:38 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK/dpdk-21.11/usertools/dpdk-devbind.py -b vfio-pci 89:01.0
2022-03-21 11:49:38 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:49:43 - test_network_sriov_stress.py - [line:172] - root - INFO - Successfuly bound VF to vfio-pci
2022-03-21 11:49:46 - sshInstance.py - [line:46] - root - INFO - /opt/APP/utility/DPDK/dpdk-21.11/usertools/dpdk-devbind.py -b vfio-pci 89:01.1
2022-03-21 11:49:47 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:49:52 - test_network_sriov_stress.py - [line:172] - root - INFO - Successfuly bound VF to vfio-pci
2022-03-21 11:49:52 - test_network_sriov_stress.py - [line:421] - root - INFO - lspci -Dnn | grep 89:01.1
2022-03-21 11:49:55 - sshInstance.py - [line:46] - root - INFO - lspci -Dnn | grep 89:01.1
2022-03-21 11:49:55 - sshInstance.py - [line:50] - root - INFO - 0000:89:01.1 Ethernet controller [0200]: Intel Corporation Ethernet Adaptive Virtual Function [8086:1889]

2022-03-21 11:50:00 - test_network_sriov_stress.py - [line:433] - root - INFO - Successfully get vf device id: 8086:1889
2022-03-21 11:50:03 - sshInstance.py - [line:46] - root - INFO - virsh net-info default | grep "Active" | awk '{print $2;}'
2022-03-21 11:50:03 - sshInstance.py - [line:50] - root - INFO - yes

2022-03-21 11:50:08 - test_network_sriov_stress.py - [line:198] - root - INFO - Default network is active? yes

2022-03-21 11:50:12 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vvcpus=5 --network network=default --hostdev pci_0000_89_01_0 --boot hd --disk patth=/home/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=55900 --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:89:01.0 is in use by driver QEMU, domain VMtest1
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest1
otherwise, please restart your installation.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:50:12 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:50:15 - test_network_sriov_stress.py - [line:230] - root - ERROR - Failed to create a VM: virt-install --name VMtest1 --ram 4096 --cpu host --vvcpus=5 --network network=default --hostdev pci_0000_89_01_0 --boot hd --disk patth=/home/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=55900 --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:89:01.0 is in use by driver QEMU, domain VMtest1
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest1
otherwise, please restart your installation.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:50:15 - test_network_sriov_stress.py - [line:231] - root - ERROR - Failed to create a VM: 
2022-03-21 11:50:19 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest1 --ram 4096 --cpu host --vvcpus=5 --network network=default --hostdev pci_0000_89_01_0 --boot hd --disk patth=/home/image/uvm2.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=55900 --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-03-21 11:50:21 - sshInstance.py - [line:71] - root - INFO - Domain creation completed.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:50:21 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:50:24 - test_network_sriov_stress.py - [line:233] - root - INFO - Successfully created a VM: VMtest1
2022-03-21 11:50:28 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vvcpus=5 --network network=default --hostdev pci_0000_89_01_1 --boot hd --disk patth=/home/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=55901 --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:89:01.1 is in use by driver QEMU, domain VMtest2
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest2
otherwise, please restart your installation.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:50:28 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:50:31 - test_network_sriov_stress.py - [line:230] - root - ERROR - Failed to create a VM: virt-install --name VMtest2 --ram 4096 --cpu host --vvcpus=5 --network network=default --hostdev pci_0000_89_01_1 --boot hd --disk patth=/home/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=55901 --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
ERROR    Requested operation is not valid: PCI device 0000:89:01.1 is in use by driver QEMU, domain VMtest2
Domain installation does not appear to have been successful.
If it was, you can restart your domain by running:
  virsh --connect qemu:///system start VMtest2
otherwise, please restart your installation.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:50:31 - test_network_sriov_stress.py - [line:231] - root - ERROR - Failed to create a VM: 
2022-03-21 11:50:35 - sshInstance.py - [line:71] - root - INFO - virt-install --name VMtest2 --ram 4096 --cpu host --vvcpus=5 --network network=default --hostdev pci_0000_89_01_1 --boot hd --disk patth=/home/image/uvm1.img,size=50,format=qcow2 --graphics vnc,listen=0.0.0.0,port=55901 --noautoconsole
WARNING  No operating system detected, VM performance may suffer. Specify an OS with --os-variant for optimal results.

Starting install...
2022-03-21 11:50:37 - sshInstance.py - [line:71] - root - INFO - Domain creation completed.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:50:37 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:50:40 - test_network_sriov_stress.py - [line:233] - root - INFO - Successfully created a VM: VMtest2
2022-03-21 11:50:43 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-03-21 11:50:43 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:50:48 - test_network_sriov_stress.py - [line:250] - root - ERROR - not found IP of VMtest1
2022-03-21 11:51:51 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest1 | grep '192.168' | awk '{print $4;}'
2022-03-21 11:51:51 - sshInstance.py - [line:50] - root - INFO - 192.168.122.128/24

2022-03-21 11:51:56 - test_network_sriov_stress.py - [line:245] - root - INFO - Successfully got VMtest1 IP: 192.168.122.128/24

2022-03-21 11:51:59 - sshInstance.py - [line:46] - root - INFO - virsh domifaddr VMtest2 | grep '192.168' | awk '{print $4;}'
2022-03-21 11:51:59 - sshInstance.py - [line:50] - root - INFO - 192.168.122.196/24

2022-03-21 11:52:04 - test_network_sriov_stress.py - [line:245] - root - INFO - Successfully got VMtest2 IP: 192.168.122.196/24

2022-03-21 11:52:04 - test_network_sriov_stress.py - [line:450] - root - INFO - ip of VM1: 192.168.122.128
2022-03-21 11:52:04 - test_network_sriov_stress.py - [line:451] - root - INFO - ip of VM1: 192.168.122.196
2022-03-21 11:52:08 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.128
PING 192.168.122.128 (192.168.122.128) 56(84) bytes of data.
64 bytes from 192.168.122.128: icmp_seq=1 ttl=64 time=0.110 ms
2022-03-21 11:52:09 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=2 ttl=64 time=0.078 ms
2022-03-21 11:52:11 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=3 ttl=64 time=0.069 ms
2022-03-21 11:52:12 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=4 ttl=64 time=0.070 ms
2022-03-21 11:52:13 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=5 ttl=64 time=0.068 ms

--- 192.168.122.128 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4091ms
rtt min/avg/max/mdev = 0.068/0.079/0.110/0.015 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:52:13 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:52:22 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:52:22 - sshInstance.py - [line:96] - root - INFO - ssh root@192.168.122.128
root@192.168.122.128's password: 
2022-03-21 11:52:31 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:52:31 - sshInstance.py - [line:96] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Sun 20 Mar 2022 11:54:17 PM EDT
  System load:  0.0                Processes:             136
  Usage of /:   37.3% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.128
  Swap usage:   0%

363 updates can be installed immediately.
221 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Sun Mar 20 23:49:14 2022 from 192.168.122.1
root@localhost:~# 
2022-03-21 11:52:40 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:52:40 - sshInstance.py - [line:96] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-03-21 11:52:46 - test_network_sriov_stress.py - [line:341] - root - INFO - successfully got vf pci id of vm1: 0000:00:05.0
2022-03-21 11:52:52 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:52:52 - sshInstance.py - [line:96] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-03-21 11:52:57 - test_network_sriov_stress.py - [line:355] - root - INFO - Successfully got vf device name of vm1: ens5
2022-03-21 11:53:03 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:53:03 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 192.168.10.12
root@localhost:~# 
2022-03-21 11:53:06 - test_network_sriov_stress.py - [line:302] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-03-21 11:53:12 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:53:12 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 down
root@localhost:~# 
2022-03-21 11:53:21 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:53:21 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 hw ether 1234567890ac
root@localhost:~# 
2022-03-21 11:53:30 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:53:30 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-03-21 11:53:39 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:53:39 - sshInstance.py - [line:96] - root - INFO - nmcli device connect ens5
Command 'nmcli' not found, but can be installed with:
apt install network-manager
root@localhost:~# 
2022-03-21 11:53:48 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:53:48 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.12  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::1034:56ff:fe78:90ac  prefixlen 64  scopeid 0x20<link>
        ether 12:34:56:78:90:ac  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 9  bytes 766 (766.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-03-21 11:53:57 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:53:57 - sshInstance.py - [line:96] - root - INFO - exit
logout
Connection to 192.168.122.128 closed.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:54:04 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.196
PING 192.168.122.196 (192.168.122.196) 56(84) bytes of data.
64 bytes from 192.168.122.196: icmp_seq=1 ttl=64 time=0.094 ms
2022-03-21 11:54:05 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.196: icmp_seq=2 ttl=64 time=0.067 ms
2022-03-21 11:54:06 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.196: icmp_seq=3 ttl=64 time=0.088 ms
2022-03-21 11:54:07 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.196: icmp_seq=4 ttl=64 time=0.073 ms
2022-03-21 11:54:08 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.196: icmp_seq=5 ttl=64 time=0.070 ms

--- 192.168.122.196 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4083ms
rtt min/avg/max/mdev = 0.067/0.078/0.094/0.010 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:54:08 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:54:17 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:54:17 - sshInstance.py - [line:96] - root - INFO - ssh root@192.168.122.196
root@192.168.122.196's password: 
2022-03-21 11:54:26 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:54:26 - sshInstance.py - [line:96] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Sun 20 Mar 2022 11:56:13 PM EDT
  System load:  0.0                Processes:             139
  Usage of /:   37.4% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.196
  Swap usage:   0%

363 updates can be installed immediately.
221 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Sun Mar 20 23:46:57 2022 from 192.168.122.1
root@localhost:~# 
2022-03-21 11:54:35 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:54:35 - sshInstance.py - [line:96] - root - INFO - lspci -Dnn | grep "8086:1889" --color=never| awk '{print $1;}' [A        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [C        [K            '
0000:00:05.0
root@localhost:~# 
2022-03-21 11:54:41 - test_network_sriov_stress.py - [line:341] - root - INFO - successfully got vf pci id of vm1: 0000:00:05.0
2022-03-21 11:54:47 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:54:47 - sshInstance.py - [line:96] - root - INFO - ls /sys/bus/pci/devices/0000\:00\:05.0/net/ --color=never
ens5
root@localhost:~# 
2022-03-21 11:54:52 - test_network_sriov_stress.py - [line:355] - root - INFO - Successfully got vf device name of vm1: ens5
2022-03-21 11:54:58 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:54:58 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 192.168.10.13
root@localhost:~# 
2022-03-21 11:55:01 - test_network_sriov_stress.py - [line:302] - root - INFO - Successfully assigned ip to ethernet: ens5
2022-03-21 11:55:07 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:55:07 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 down
root@localhost:~# 
2022-03-21 11:55:16 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:55:16 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 hw ether 1234567890ab
root@localhost:~# 
2022-03-21 11:55:25 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:55:25 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5 up
root@localhost:~# 
2022-03-21 11:55:34 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:55:34 - sshInstance.py - [line:96] - root - INFO - nmcli device connect ens5
Command 'nmcli' not found, but can be installed with:
apt install network-manager
root@localhost:~# 
2022-03-21 11:55:43 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:55:43 - sshInstance.py - [line:96] - root - INFO - ifconfig ens5
ens5: flags=4163<UP,BROADCAST,RUNNING,MULTICAST>  mtu 1500
        inet 192.168.10.13  netmask 255.255.255.0  broadcast 192.168.10.255
        inet6 fe80::1034:56ff:fe78:90ab  prefixlen 64  scopeid 0x20<link>
        ether 12:34:56:78:90:ab  txqueuelen 1000  (Ethernet)
        RX packets 0  bytes 0 (0.0 B)
        RX errors 0  dropped 0  overruns 0  frame 0
        TX packets 9  bytes 766 (766.0 B)
        TX errors 0  dropped 0 overruns 0  carrier 0  collisions 0
root@localhost:~# 
2022-03-21 11:55:52 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:55:52 - sshInstance.py - [line:96] - root - INFO - exit
logout
Connection to 192.168.122.196 closed.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:55:55 - test_network_sriov_stress.py - [line:469] - root - INFO - Ping two vms IP from SUT:
2022-03-21 11:55:59 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.12
PING 192.168.10.12 (192.168.10.12) 56(84) bytes of data.
64 bytes from 192.168.10.12: icmp_seq=1 ttl=64 time=0.104 ms
2022-03-21 11:56:00 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=2 ttl=64 time=0.040 ms
2022-03-21 11:56:01 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=3 ttl=64 time=0.036 ms
2022-03-21 11:56:02 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=4 ttl=64 time=0.033 ms
2022-03-21 11:56:03 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.12: icmp_seq=5 ttl=64 time=0.034 ms

--- 192.168.10.12 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4078ms
rtt min/avg/max/mdev = 0.033/0.049/0.104/0.027 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:56:03 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:56:10 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=0.094 ms
2022-03-21 11:56:11 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.045 ms
2022-03-21 11:56:12 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.028 ms
2022-03-21 11:56:13 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.038 ms
2022-03-21 11:56:14 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.035 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4102ms
rtt min/avg/max/mdev = 0.028/0.048/0.094/0.023 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:56:14 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:56:17 - test_network_sriov_stress.py - [line:477] - root - INFO - ping VMtest2 IP from VMtest1
2022-03-21 11:56:21 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.122.128
PING 192.168.122.128 (192.168.122.128) 56(84) bytes of data.
64 bytes from 192.168.122.128: icmp_seq=1 ttl=64 time=0.083 ms
2022-03-21 11:56:22 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=2 ttl=64 time=0.076 ms
2022-03-21 11:56:23 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=3 ttl=64 time=0.063 ms
2022-03-21 11:56:24 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=4 ttl=64 time=0.072 ms
2022-03-21 11:56:25 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.122.128: icmp_seq=5 ttl=64 time=0.065 ms

--- 192.168.122.128 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4095ms
rtt min/avg/max/mdev = 0.063/0.071/0.083/0.007 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:56:25 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:56:34 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:56:34 - sshInstance.py - [line:96] - root - INFO - ssh root@192.168.122.128
root@192.168.122.128's password: 
2022-03-21 11:56:43 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:56:43 - sshInstance.py - [line:96] - root - INFO - 
Welcome to Ubuntu 20.04 LTS (GNU/Linux 5.4.0-26-generic x86_64)
 * Documentation:  https://help.ubuntu.com
 * Management:     https://landscape.canonical.com
 * Support:        https://ubuntu.com/advantage
  System information as of Sun 20 Mar 2022 11:58:30 PM EDT
  System load:  0.0                Processes:             129
  Usage of /:   37.3% of 19.07GB   Users logged in:       0
  Memory usage: 4%                 IPv4 address for ens3: 192.168.122.128
  Swap usage:   0%                 IPv4 address for ens5: 192.168.10.12

363 updates can be installed immediately.
221 of these updates are security updates.
To see these additional updates run: apt list --upgradable

The list of available updates is more than a week old.
To check for new updates run: sudo apt update
Last login: Sun Mar 20 23:54:18 2022 from 192.168.122.1
root@localhost:~# 
2022-03-21 11:56:50 - sshInstance.py - [line:71] - root - INFO - ping -c 5 192.168.10.13
PING 192.168.10.13 (192.168.10.13) 56(84) bytes of data.
64 bytes from 192.168.10.13: icmp_seq=1 ttl=64 time=0.273 ms
2022-03-21 11:56:51 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=2 ttl=64 time=0.046 ms
2022-03-21 11:56:52 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=3 ttl=64 time=0.030 ms
2022-03-21 11:56:53 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=4 ttl=64 time=0.036 ms
2022-03-21 11:56:54 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.13: icmp_seq=5 ttl=64 time=0.025 ms

--- 192.168.10.13 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4079ms
rtt min/avg/max/mdev = 0.025/0.082/0.273/0.095 ms
root@localhost:~# 
2022-03-21 11:56:54 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:57:03 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:57:03 - sshInstance.py - [line:96] - root - INFO - exit
logout
Connection to 192.168.122.128 closed.
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:57:06 - test_network_sriov_stress.py - [line:485] - root - INFO - ping VMtest2 IP from VMtest1
2022-03-21 11:57:10 - sshInstance.py - [line:71] - root - INFO - ping -c 5 -I 192.168.10.10 192.168.10.11
PING 192.168.10.11 (192.168.10.11) from 192.168.10.10 : 56(84) bytes of data.
64 bytes from 192.168.10.11: icmp_seq=1 ttl=64 time=0.060 ms
2022-03-21 11:57:11 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=2 ttl=64 time=0.040 ms
2022-03-21 11:57:12 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=3 ttl=64 time=0.047 ms
2022-03-21 11:57:13 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=4 ttl=64 time=0.039 ms
2022-03-21 11:57:14 - sshInstance.py - [line:71] - root - INFO - 64 bytes from 192.168.10.11: icmp_seq=5 ttl=64 time=0.040 ms

--- 192.168.10.11 ping statistics ---
5 packets transmitted, 5 received, 0% packet loss, time 4086ms
rtt min/avg/max/mdev = 0.039/0.045/0.060/0.007 ms
root@npx-sut3-ice-d-auto:~# 
2022-03-21 11:57:14 - sshInstance.py - [line:94] - root - INFO - --cmd is end--
2022-03-21 11:57:20 - sshInstance.py - [line:46] - root - INFO - echo 0 > /sys/class/net/enp137s0f0/device/sriov_numvfs
2022-03-21 11:57:23 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:57:28 - test_network_sriov_stress.py - [line:264] - root - INFO - Successfully removed VFs from PF.
2022-03-21 11:57:31 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest1
2022-03-21 11:57:31 - sshInstance.py - [line:50] - root - INFO - Domain VMtest1 destroyed


2022-03-21 11:57:36 - test_network_sriov_stress.py - [line:277] - root - INFO - Successfully shut down the VM: VMtest1
2022-03-21 11:57:39 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest1
2022-03-21 11:57:39 - sshInstance.py - [line:50] - root - INFO - Domain VMtest1 has been undefined


2022-03-21 11:57:44 - test_network_sriov_stress.py - [line:287] - root - INFO - Successfully removed the VM: VMtest1
2022-03-21 11:57:47 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest2
2022-03-21 11:57:47 - sshInstance.py - [line:50] - root - INFO - Domain VMtest2 destroyed


2022-03-21 11:57:52 - test_network_sriov_stress.py - [line:277] - root - INFO - Successfully shut down the VM: VMtest2
2022-03-21 11:57:55 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest2
2022-03-21 11:57:55 - sshInstance.py - [line:50] - root - INFO - Domain VMtest2 has been undefined


2022-03-21 11:58:00 - test_network_sriov_stress.py - [line:287] - root - INFO - Successfully removed the VM: VMtest2
.2022-03-21 11:58:03 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest2
2022-03-21 11:58:03 - sshInstance.py - [line:50] - root - INFO - error: failed to get domain 'VMtest2'


2022-03-21 11:58:08 - test_network_sriov_stress.py - [line:277] - root - INFO - Successfully shut down the VM: VMtest2
2022-03-21 11:58:11 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest2
2022-03-21 11:58:12 - sshInstance.py - [line:50] - root - INFO - error: failed to get domain 'VMtest2'


2022-03-21 11:58:17 - test_network_sriov_stress.py - [line:290] - root - ERROR - Failed to removed the VM VMtest2
2022-03-21 11:58:20 - sshInstance.py - [line:46] - root - INFO - virsh destroy VMtest1
2022-03-21 11:58:20 - sshInstance.py - [line:50] - root - INFO - error: failed to get domain 'VMtest1'


2022-03-21 11:58:25 - test_network_sriov_stress.py - [line:277] - root - INFO - Successfully shut down the VM: VMtest1
2022-03-21 11:58:28 - sshInstance.py - [line:46] - root - INFO - virsh undefine VMtest1
2022-03-21 11:58:28 - sshInstance.py - [line:50] - root - INFO - error: failed to get domain 'VMtest1'


2022-03-21 11:58:33 - test_network_sriov_stress.py - [line:290] - root - ERROR - Failed to removed the VM VMtest1
2022-03-21 11:58:36 - sshInstance.py - [line:46] - root - INFO - echo 0 > /sys/class/net/enp137s0f0/device/sriov_numvfs
2022-03-21 11:58:36 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:58:44 - sshInstance.py - [line:46] - root - INFO - ifconfig enp137s0f0 0
2022-03-21 11:58:44 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:58:52 - sshInstance.py - [line:46] - root - INFO - ifconfig ens801f0 0
2022-03-21 11:58:52 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:59:00 - sshInstance.py - [line:46] - root - INFO - dmesg > /tmp/dmesg.log
2022-03-21 11:59:00 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:59:05 - test_network_sriov_stress.py - [line:82] - root - INFO - generate dmesg.log file on sut /tmp/dmesg.log
2022-03-21 11:59:05 - test_network_sriov_stress.py - [line:84] - root - INFO - copy dmesg.log from sut1 /tmp dir to controller dir /root/testconfig/testlogs/2022-03-21/test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[uvm2.img-uvm1.img]_11_10_35
2022-03-21 11:59:08 - sshInstance.py - [line:46] - root - INFO - rm -f /tmp/dmesg.log
2022-03-21 11:59:08 - sshInstance.py - [line:50] - root - INFO - 
2022-03-21 11:59:13 - test_network_sriov_stress.py - [line:86] - root - INFO - rm dmesg.log on sut1 /tmp dir
2022-03-21 11:59:13 - sshInstance.py - [line:39] - root - INFO - closing ssh client on SUT
2022-03-21 11:59:13 - NicTestFixture.py - [line:34] - root - INFO - ============================ end of run test_PI_Networking_SRIOV_StressDisableEnableFor5TimesWhenVFIsAssigned_VM_Guest[uvm2.img-uvm1.img] case ============================
                  [100%]

=============================== warnings summary ===============================
npxtester/Tests/testNics/test_iperf_client_4hour.py:70
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_iperf_client_4hour.py:70: PytestUnknownMarkWarning: Unknown pytest.mark.iperf_tcp - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.iperf_tcp

npxtester/Tests/testNics/test_iperf_client_4hour.py:108
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_iperf_client_4hour.py:108: PytestUnknownMarkWarning: Unknown pytest.mark.iperf_udp - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.iperf_udp

npxtester/Tests/testNics/test_mtu.py:69
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_mtu.py:69: PytestUnknownMarkWarning: Unknown pytest.mark.two_sut_CVL - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.two_sut_CVL

npxtester/Tests/testNics/test_mtu.py:70
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_mtu.py:70: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_mtu.py:109
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_mtu.py:109: PytestUnknownMarkWarning: Unknown pytest.mark.two_sut_CVL - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.two_sut_CVL

npxtester/Tests/testNics/test_mtu.py:110
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_mtu.py:110: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_network_sriov_stress.py:382
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_network_sriov_stress.py:382: PytestUnknownMarkWarning: Unknown pytest.mark.nicSriov5times - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSriov5times

npxtester/Tests/testNics/test_network_sriov_stress.py:501
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_network_sriov_stress.py:501: PytestUnknownMarkWarning: Unknown pytest.mark.nicSriov2hours - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSriov2hours

npxtester/Tests/testNics/test_network_sriov_stress.py:800
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_network_sriov_stress.py:800: PytestUnknownMarkWarning: Unknown pytest.mark.SriovVfIperf - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.SriovVfIperf

npxtester/Tests/testNics/test_nics.py:86
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:86: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:113
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:113: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:146
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:146: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:217
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:217: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

npxtester/Tests/testNics/test_nics.py:243
  /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/Tests/testNics/test_nics.py:243: PytestUnknownMarkWarning: Unknown pytest.mark.nicSimple - is this a typo?  You can register custom marks to avoid this warning - for details, see https://docs.pytest.org/en/stable/mark.html
    @pytest.mark.nicSimple

-- Docs: https://docs.pytest.org/en/stable/warnings.html
- generated html file: file:///root/testconfig/testlogs/HtmlReport/nic_sriov5times_case_report.html -
========== 1 passed, 16 deselected, 14 warnings in 2917.83s (0:48:37) ==========
[8mha:////4LeiMYyvaTAszhIQ10pdIeDepJ20hGjpoNTSDQ54vTlOAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIi0SCEqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDyFBu95/avQGMKkS1vgAAAA==[0m[Pipeline] }
[8mha:////4I/tRoD/KV6lJ76TTEJ8vusLD5mbZiDJ1z7tZBIwhHThAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOKCQKJBqWgtN7zAJMY4se6C7eBUvIiv8QcsIlGxxWpnm3m9YRk81OwNdpp6S6GxOLjR5IWJfX91nLDjCzZMgZ1GqZPkVp9mlBw1zClKWAhYaWocB0smwlp06qEqp8hU5+jzdxRQ2vYOTyhENkfl4xcmD5vb2Aamn+evAKZhiFDut7l2hw9mQDtDvgAAAA==[0m[Pipeline] // stage
[8mha:////4CAP7WQdC2FMpNczqgZSKdPjM+Zu6Lz59I8VEE3/AlUOAAAAph+LCAAAAAAAAP9tjTEOwjAQBM9BKWgpeYQjBB2iorXc8AITG+PEugv2haTiRXyNPxCIRMVWOyut5vmCMic4UPKycdgGzHWQXez91ORAqb1EGmRDZ1kTZopOajdosu44oyZ2MEcUsFCwdFhHygE9w0o15m6qaNBXJ07TtldQBHuDBwg1mdkk/sKYYH3tbSb8ef4KYOwYxI6h2G4+x/INOibpDcEAAAA=[0m[Pipeline] stage
[8mha:////4Db8sOlSRxZnVxb7xJBDSAteoF0PgC9lkQldzKIJFwYiAAAAph+LCAAAAAAAAP9tjTEOwjAUQ3+LOrAycohUwIiYukZZOEFoQkgb/V9+UtqJE3E17kBLJSY8WLYl673eUESGE7ETjcXWY6y96ELvpiQG4vYaaBANXURNGClYoeygyNhqqYqShUVZDisJa4t1oOjRJdjIRj90GTS68px42o4Scm/u8IRMTuSkOX3LyLC99SYS/jh/ATB2CfLDbrb9/Cw+2YMO4MIAAAA=[0m[Pipeline] { (collect logs)
[8mha:////4LN/0LMxu1hyybTBeE6CHwDx82bv/lkvxFk3cdgHTSuWAAAAox+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIikRJR0VppeIFJjLFj3Tm2g1PxIr7GHwhEomKL1c4283zBOgY4ctBoFfWGYmvQu1HPCzOH/uo4o+ULtkyRncJG5YY7dVqw4aRgSVHCSsBGUes4GtIJtsLKu6ycJF2dU5i/g4DSdAM8oBCzOcmQvjAF2N3GLjL9PH8FMPkEZb3/VO3favo5XL4AAAA=[0m[Pipeline] sh
+ sed -i $'s/\r//g' /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/log_analyzer/log_analyzer.sh
[8mha:////4FgvIflgycWoJq7MsZycbV9d7HENRG2WSX14oF+Me/GuAAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiAR2iSmu54QUmNsaJdWdsh6TiRXyNPxCIRMUWq51t5vmCZYpw5GixNdQ5So3D4Hs7LRw4dhfPA7Z8xoYpsTcozSBZm3pGydnAnKKEhYCVocZzcmQzrEWr7qryimx1ynH6DgJKp2/wgEJM5qxi/sIYYXPtdWL6ef4KYAwZyt32U/vwBq1seBO+AAAA[0m[Pipeline] sh
+ chmod a+x /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/log_analyzer/log_analyzer.sh
[8mha:////4P37N80+NRbzyFlga2j1WDon6H/4VmnQwrRysqQpQX/3AAAApB+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIigagQVVrLDS8wsTFOrDtjOyQVL+Jr/IFAJCq2WO1sM88XLFOEI0eLraHOUWocBt/baeHAsbt4HrDlMzZMib1BaQbJ2tQzSs4G5hQlLASsDDWekyObYS1adVeVV2SrU47TdxBQOn2DBxRiMmcV8xfGCJtrrxPTz/NXAGPIUO62n9qHN+xdYwq+AAAA[0m[Pipeline] sh
+ sh /root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/log_analyzer/log_analyzer.sh
/root/.jenkins/workspace/icx-d-test-nic-sriov-5times/npxtester/log_analyzer/log_analyzer.sh: line 7: [: too many arguments
[8mha:////4P3ih/vRXCzaXCkS1CQDBQXrb3I4EZcCr0NGGH+tKlR4AAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIigUSDqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDyFDu9p/avgEjVhSLvgAAAA==[0m[Pipeline] }
[8mha:////4ISV0Ptmba0km8a80Gq+UClogdZnPDxCJXroOoeI5sULAAAApR+LCAAAAAAAAP9tjTEOwjAQBC9BFLSUPOIiREGBqNJabniBiY1xYt0Z2yGpeBFf4w8EIlGxxWpnm3m+YJkiHDlabA11jlLjMPjeTgsHjt3F84Atn7FhSuwNSjNI1qaeUXI2MKcoYSFgZajxnBzZDGvRqruqvCJbnXKcvoOA0ukbPKAQkzmrmL8wRthce52Yfp6/AhhDyFDu9p/avgE81EHZvgAAAA==[0m[Pipeline] // stage
[8mha:////4LmA5LOJ1yGOO5YQ6RYn+/szIsYEkfxAArsUKSFs0Xa0AAAApB+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMIpoEBCVLRWGl5gEmOcWHfBvpBUvIiv8QciIlGx1c4083pjmSKOHJ1qLLWeUuVVF3o3PTVwbK+BB9XwRVVMiYNVpR1Kru1pxpLFYl6WY6GxslQFTp6cYK0b8zBFMOSKs8TJHTRyX9/xRKanspgoXxgjNre+Tky/zt8Axq4T5Nu9INt9AE/B/a+9AAAA[0m[Pipeline] }
[8mha:////4KVKO0UZMawo8C9tjXN4L1+ryWbU4zmvD2sDlsKoyYtXAAAAox+LCAAAAAAAAP9tjTESgjAQRT84FraWHiIUVI5jZZuh8QQRYgxkdjFZhMoTeTXvICMzVv7qv9e81xvrFHHk6FRrqfOUaq/6MLj5qZFjdw08qpYvqmZKHKyq7FhxY08LViwWy7IcK42NpTpw8uQEW92ahymCIVecJc7uoJH75o4nMj2XxUT5whSxuw1NYvp1/gYw9b0gL/eCrPwAXH35DL0AAAA=[0m[Pipeline] // node
[8mha:////4A8vvhfR6oegt+2XAbvn3Z0Fx6j3lb1QUh2abilCs+OGAAAAox+LCAAAAAAAAP9tjTEOwjAQBDdBFLSUPMJBiA5R0VppeIFJjHFi3QX7QlLxIr7GH4iIRMVWO9PM641lijhydKqx1HpKlVdd6N301MCxvQYeVMMXVTElDlaVdii5tqcZSxaLeVmOhcbKUhU4eXKCtW7MwxTBkCvOEid30Mh9fccTmZ7KYqJ8YYzY3Po6Mf06fwMYu06Q77eCbPcBAx/ra70AAAA=[0m[Pipeline] End of Pipeline
Finished: SUCCESS
